<html>
                    <head>
                    <title>Individual alignments and phylogenetic tree for taxon lepidosauria excl. toxicofera</title>
                    <style>
                        body { font-family: "Open Sans", Arial; }
                    </style>
                    <script type="text/javascript">
                    <!--
                        function toggle_visibility(id) {
                            var e = document.getElementById(id);
                            if(e.style.display == 'block')
                                e.style.display = 'none';
                            else
                                e.style.display = 'block'; }
                    //-->
                    </script>
                    </head>
                    <body>
                    <h1>lepidosauria excl. toxicofera</h1>
                    <table border="1"><tr><td><a id=XP_028599647.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028599647.1/" target="_blank">XP_028599647.1</a></a></td><td>XP_028599647</td><td>PDGF-D</td><td>vascular endothelial growth factor C isoform X2 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028599647.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028599647.1');">alignment</a><div id="align_XP_028599647.1" style="display:none"><pre>


           1                                               50
XP_028599647.1  MHLLGFFSLGCYLAAGAVVLLGAREPATAAAYESGQGYYEEEPDVGDAKT
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028599647.1  HASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHNREQSSFD
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028599647.1  ARSEDSNPVKFAAAHYNPEILKSIDNEWRKTQCMPREVCVDVGKEYGATT
VEGF-C          SRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVAT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXX

           151                                            200
XP_028599647.1  NTFFKPPCVSIYRCGGCCNSEGLQCMNISTSFISKTLFEITVPLSHGPKP
VEGF-C          NTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKP
VEGFC_signature XXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028599647.1  VTISFANHTSCRCMSKLDVYRQVHSIIRRSLSVTQTQCHVANKTCPKNHI
VEGF-C          VTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYM
VEGFC_signature XXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028599647.1  WNNQICRCLAQHDFSFSSYLGDTDTAEGYHDICGPNKELDEETCQCVCKG
VEGF-C          WNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVCRA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028599647.1  GVRPSSCGPHKELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCVCKRTC
VEGF-C          GLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028599647.1  PKHQPLNSAKCICECVESPNKCFLKGKRFNHQTCSCYRPPCTVRTKRCEP
VEGF-C          PRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                422
XP_028599647.1  GFYYSEEVCRCVPTYWKRPLIN
VEGF-C          GFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028599646.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028599646.1/" target="_blank">XP_028599646.1</a></a></td><td>XP_028599646</td><td>PDGF-D</td><td>vascular endothelial growth factor C isoform X1 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028599646.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028599646.1');">alignment</a><div id="align_XP_028599646.1" style="display:none"><pre>


           1                                               50
XP_028599646.1  MHLLGFFSLGCYLAAGAVVLLGAREPATAAAYESGQGYYEEEPDVGDAKQ
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAG.EAT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028599646.1  THASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHNREQSSF
VEGF-C          AYASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028599646.1  DARSEDSNPVKFAAAHYNPEILKSIDNEWRKTQCMPREVCVDVGKEYGAT
VEGF-C          NSRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXX

           151                                            200
XP_028599646.1  TNTFFKPPCVSIYRCGGCCNSEGLQCMNISTSFISKTLFEITVPLSHGPK
VEGF-C          TNTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPK
VEGFC_signature XXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028599646.1  PVTISFANHTSCRCMSKLDVYRQVHSIIRRSLSVTQTQCHVANKTCPKNH
VEGF-C          PVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNY
VEGFC_signature XXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028599646.1  IWNNQICRCLAQHDFSFSSYLGDTDTAEGYHDICGPNKELDEETCQCVCK
VEGF-C          MWNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVCR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028599646.1  GGVRPSSCGPHKELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCVCKRT
VEGF-C          AGLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028599646.1  CPKHQPLNSAKCICECVESPNKCFLKGKRFNHQTCSCYRPPCTVRTKRCE
VEGF-C          CPRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                 423
XP_028599646.1  PGFYYSEEVCRCVPTYWKRPLIN
VEGF-C          PGFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF50945.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF50945.1/" target="_blank">GCF50945.1</a></a></td><td>GCF50945</td><td>PDGF-D</td><td>hypothetical protein parPi_0012082 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF50945.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF50945.1');">alignment</a><div id="align_GCF50945.1" style="display:none"><pre>


           1                                               50
GCF50945.1      MHLLGVLSLGCSLAAATLGLLGAREPATAAAYESGQGYYEEEPELLAEAK
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEP..DAGEA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF50945.1      QAYASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHSREQSS
VEGF-C          TAYASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQAN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF50945.1      FDARSEDSNPIKFAAAHYNADILKSIDNEWRKTQCMPREVCVDVGKEFGA
VEGF-C          LNSRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXX

           151                                            200
GCF50945.1      TTNTFFKPPCVSIYRCGGCCNSEGLQCMNISTSYISKTLFEITVPLSHGP
VEGF-C          ATNTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGP
VEGFC_signature XXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
GCF50945.1      KPVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPLTQQQCHVANKTCPKS
VEGF-C          KPVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTN
VEGFC_signature XXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF50945.1      YIWNNRICRCLAQHDFSFSSHTGDTDSAEGFHDICGPNKELDEETCQCVC
VEGF-C          YMWNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
GCF50945.1      KGGVRPSSCGAHKELDRTSCQCTCKNKLLPSSCGPNRDFDEERCHCVCKR
VEGF-C          RAGLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
GCF50945.1      TCPKHQPLNPAKCICECVESPNKCFLKGKRFHQQTCSCYRPPCTVRMKRC
VEGF-C          TCPRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKAC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                  424
GCF50945.1      ESGFYYSEEVCRCVPIYWKRPLIN
VEGF-C          EPGFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015283812.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015283812.1/" target="_blank">XP_015283812.1</a></a></td><td>XP_015283812</td><td>PDGF-D</td><td>PREDICTED: vascular endothelial growth factor C [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015283812.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015283812.1');">alignment</a><div id="align_XP_015283812.1" style="display:none"><pre>


           1                                               50
XP_015283812.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MVSEDGIDHFLRA
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015283812.1  YASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHSREQSSFI
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015283812.1  ARSEDSNPIKFAAAHYNADILKSIDNEWRKTQCMPREVCVDVGKEFGATT
VEGF-C          SRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVAT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXX

           151                                            200
XP_015283812.1  NTFFKPPCVSIYRCGGCCNSEGLQCMNISTSYISKTLFEITVPLSHGPKP
VEGF-C          NTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKP
VEGFC_signature XXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015283812.1  VTISFANHTSCRCMSKLDVYRQVHSIIRRSLPLTQPQCHVANKTCPKSYI
VEGF-C          VTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYM
VEGFC_signature XXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015283812.1  WNNHMCRCLAQHDFSFSSHLGDTDSTEGFHDICGPNKELDEETCQCVCKG
VEGF-C          WNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVCRA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015283812.1  GVRPSSCGAHKELDRTSCECTCKNKLLPSSCGPNKDFDEERCHCVCKKTC
VEGF-C          GLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015283812.1  PKHQPLNPAKCICECVESPNRCFLKGKRFHQQTCSCYRPPCTVRTKRCES
VEGF-C          PRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                422
XP_015283812.1  GFYYSEEVCRCVPIYWKRPLIN
VEGF-C          GFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028599648.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028599648.1/" target="_blank">XP_028599648.1</a></a></td><td>XP_028599648</td><td>PDGF-D</td><td>vascular endothelial growth factor C isoform X3 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028599648.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028599648.1');">alignment</a><div id="align_XP_028599648.1" style="display:none"><pre>


           1                                               50
XP_028599648.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MVVGCKCRPPRDQQT
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028599648.1  HASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHNREQSSFD
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028599648.1  ARSEDSNPVKFAAAHYNPEILKSIDNEWRKTQCMPREVCVDVGKEYGATT
VEGF-C          SRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVAT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXX

           151                                            200
XP_028599648.1  NTFFKPPCVSIYRCGGCCNSEGLQCMNISTSFISKTLFEITVPLSHGPKP
VEGF-C          NTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKP
VEGFC_signature XXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028599648.1  VTISFANHTSCRCMSKLDVYRQVHSIIRRSLSVTQTQCHVANKTCPKNHI
VEGF-C          VTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYM
VEGFC_signature XXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028599648.1  WNNQICRCLAQHDFSFSSYLGDTDTAEGYHDICGPNKELDEETCQCVCKG
VEGF-C          WNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVCRA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028599648.1  GVRPSSCGPHKELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCVCKRTC
VEGF-C          GLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028599648.1  PKHQPLNSAKCICECVESPNKCFLKGKRFNHQTCSCYRPPCTVRTKRCEP
VEGF-C          PRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                422
XP_028599648.1  GFYYSEEVCRCVPTYWKRPLIN
VEGF-C          GFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028599649.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028599649.1/" target="_blank">XP_028599649.1</a></a></td><td>XP_028599649</td><td>PDGF-D</td><td>vascular endothelial growth factor C isoform X4 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028599649.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028599649.1');">alignment</a><div id="align_XP_028599649.1" style="display:none"><pre>


           1                                               50
XP_028599649.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MVVGCKCRPPRDQT
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028599649.1  HASKELEEQLRSVSSVDELMTVLYPEYWKMFKCQLRKGSWQHNREQSSFD
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028599649.1  ARSEDSNPVKFAAAHYNPEILKSIDNEWRKTQCMPREVCVDVGKEYGATT
VEGF-C          SRTEET..IKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVAT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXX

           151                                            200
XP_028599649.1  NTFFKPPCVSIYRCGGCCNSEGLQCMNISTSFISKTLFEITVPLSHGPKP
VEGF-C          NTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKP
VEGFC_signature XXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028599649.1  VTISFANHTSCRCMSKLDVYRQVHSIIRRSLSVTQTQCHVANKTCPKNHI
VEGF-C          VTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYM
VEGFC_signature XXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028599649.1  WNNQICRCLAQHDFSFSSYLGDTDTAEGYHDICGPNKELDEETCQCVCKG
VEGF-C          WNNHICRCLAQEDFMFSSDAGD.DSTDGFHDICGPNKELDEETCQCVCRA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028599649.1  GVRPSSCGPHKELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCVCKRTC
VEGF-C          GLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028599649.1  PKHQPLNSAKCICECVESPNKCFLKGKRFNHQTCSCYRPPCTVRTKRCEP
VEGF-C          PRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                422
XP_028599649.1  GFYYSEEVCRCVPTYWKRPLIN
VEGF-C          GFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF40285.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF40285.1/" target="_blank">GCF40285.1</a></a></td><td>GCF40285</td><td>PDGF-D</td><td>hypothetical protein parPi_0000734 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF40285.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF40285.1');">alignment</a><div id="align_GCF40285.1" style="display:none"><pre>


           1                                               50
GCF40285.1      ~~~~~~~~~~~~~~~~~MYKLWTTVNVFMVSFLHLLQGSESD.SVKKPSL
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF40285.1      SASEWSEQQIRKAASLEELLRITHSEDWKLWKCRLK....LKSVATLDSR
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF40285.1      SASHRSTRFAAAFYDIEILKVIDEEWQRTQCMPRETCVEVAKELGTTTNK
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
GCF40285.1      FFKPPCVNVYRCGGCCNEESLSCTNTSTSYVSKMLFEISVPLTNVPDPIQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
GCF40285.1      VKIANHTGCKCIS..NVQHQSYSIIRRSVPYSEEDRCSHFKKLCHHGWIW
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLP.QCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF40285.1      DVHKCECVENEDHPSRREG..LPPLAELAMCAPNMDFDEENCECICKW..
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
GCF40285.1      ...KCTGNLFQNKENCTCYLCTENQDSCFQKHKTFNAETCSCEDK..CPF
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
GCF40285.1      QARTCP..TARPACSRHCRCLRGGRSPHGSQSRENP~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
GCF40285.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015273092.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015273092.1/" target="_blank">XP_015273092.1</a></a></td><td>XP_015273092</td><td>PDGF-D</td><td>PREDICTED: vascular endothelial growth factor D [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015273092.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015273092.1');">alignment</a><div id="align_XP_015273092.1" style="display:none"><pre>


           1                                               50
XP_015273092.1  ~~~~~~~~~~~~~~~~~~~~MYKLWTTVNVFMVSLLHLLQGYGSVKRASL
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015273092.1  SALEWSEQQIRKAASLEELLRITHSEDWKLWKCRLK....LKSVATLDSR
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015273092.1  SASHRSTRFAAAFYDIEILKVIDEEWQRTQCVPRETCVEVAKELGTTTNK
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015273092.1  FFKPPCVNVYRCGGCCNEESLSCMNTSTSYVSKMLFEISVPLTNVPDPIQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015273092.1  VKIANHTDCKCIS..NGQHQPYSIIRRSVPYSEEDRCSHLKKLCHNGWIW
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLP.QCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015273092.1  DINKCECVQNEDHPSRREG..LPPLAELAMCAPNMDFDEENCECVCKW..
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015273092.1  ...KCTGNLFQNKENCSCYLCTENQDSCFQKHKTFNAETCSCEDK..CPF
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015273092.1  QARTCP..TARPACSRHCRCLRGGRGHHGSQSKENP~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_015273092.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028583464.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028583464.1/" target="_blank">XP_028583464.1</a></a></td><td>XP_028583464</td><td>PDGF-D</td><td>vascular endothelial growth factor D isoform X2 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028583464.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028583464.1');">alignment</a><div id="align_XP_028583464.1" style="display:none"><pre>


           1                                               50
XP_028583464.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MLGPIDGETTEEVLEERTSL
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028583464.1  SALEWSEQQIRKAASLEELLRITHSEDWKLWKCRLKLKGMAALDS....R
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028583464.1  SASHRSTRFAAAFYDIEILKVIDEEWQRTQCVPRETCVEVAKELGTATNK
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_028583464.1  FFKPPCVNVYRCGGCCNEESLSCVNTSTSYISKMLFEISVPLTNVPDPIQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028583464.1  VKIANHTGCKCLT..NAQRPSYTIIRRSVQYPEEDSCPQINKLCRSGWMW
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLP.QCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028583464.1  DSDKCECVVDKDHPGRREG..LPPLAELAMCAPNMNFDEENCECICKW..
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028583464.1  ...KCTGNLFQNKENCSCYLCTENQDSCLQKHKEFNAETCSCEDKCPFQA
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKR.TCP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028583464.1  RMCPTARPACSRHCCRPRGGRSPHGSQSRETP~~~~~~~~~~~~~~~~~~
VEGF-C          RNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401               421
XP_028583464.1  ~~~~~~~~~~~~~~~~~~~~~
VEGF-C          FSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028583463.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028583463.1/" target="_blank">XP_028583463.1</a></a></td><td>XP_028583463</td><td>PDGF-D</td><td>vascular endothelial growth factor D isoform X1 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028583463.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028583463.1');">alignment</a><div id="align_XP_028583463.1" style="display:none"><pre>


           1                                               50
XP_028583463.1  ~~~~~~~~~~~~~~~~~~MYKPWTTVNIFIVSFLHLLQGSEHGSVKRTSL
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028583463.1  SALEWSEQQIRKAASLEELLRITHSEDWKLWKCRLKLKGMAALDS....R
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028583463.1  SASHRSTRFAAAFYDIEILKVIDEEWQRTQCVPRETCVEVAKELGTATNK
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_028583463.1  FFKPPCVNVYRCGGCCNEESLSCVNTSTSYISKMLFEISVPLTNVPDPIQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028583463.1  VKIANHTGCKCLT..NAQRPSYTIIRRSVQYPEEDSCPQINKLCRSGWMW
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLP.QCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028583463.1  DSDKCECVVDKDHPGRREG..LPPLAELAMCAPNMNFDEENCECICKW..
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028583463.1  ...KCTGNLFQNKENCSCYLCTENQDSCLQKHKEFNAETCSCEDKCPFQA
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKR.TCP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028583463.1  RMCPTARPACSRHCCRPRGGRSPHGSQSRETP~~~~~~~~~~~~~~~~~~
VEGF-C          RNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401               421
XP_028583463.1  ~~~~~~~~~~~~~~~~~~~~~
VEGF-C          FSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADJ67670.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADJ67670.1/" target="_blank">ADJ67670.1</a></a></td><td>ADJ67670</td><td>VEGF-D</td><td>vascular endothelial growth factor C, partial [Christinus marmoratus]</td><td>VEGF-C</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADJ67670.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADJ67670.1');">alignment</a><div id="align_ADJ67670.1" style="display:none"><pre>


           1                                               50
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADJ67670.1      ~~~~~~~~~~~~~~~~NSEGLQCMNISTSYISKTLFEITVPLSHGPKPVT
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADJ67670.1      ISFANHTSCRCMS~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADJ67670.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADJ67669.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADJ67669.1/" target="_blank">ADJ67669.1</a></a></td><td>ADJ67669</td><td>PDGF-B</td><td>vascular endothelial growth factor D, partial [Christinus marmoratus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADJ67669.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADJ67669.1');">alignment</a><div id="align_ADJ67669.1" style="display:none"><pre>


           1                                               50
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~TQCVPRETCVDVAKDLGTTTNK
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADJ67669.1      FFKPPCVNVYRCGGCCNEESRSCMNT~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADJ67669.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028597747.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028597747.1/" target="_blank">XP_028597747.1</a></a></td><td>XP_028597747</td><td>PDGF-B</td><td>snake venom vascular endothelial growth factor toxin HF-like isoform X3 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028597747.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028597747.1');">alignment</a><div id="align_XP_028597747.1" style="display:none"><pre>


           1                                               50
XP_028597747.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028597747.1  ~~~~~~~~~~~MPSSADSGAGRAWTSLLQHRSSSRSRTKRMPVPGYLVAI
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028597747.1  ALLCAQPVLTSEQP..AGVVPFEEVWSRSYCRARETLVDILSEFPHEAEH
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_028597747.1  IFKPPCVPLWRCAGCCGDESLECIAVETRTIDLQVVRVSPILG.TTQQEE
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028597747.1  MKFVEHTRCSCRPRRKRLKGESSCKLPGAGRPEAQHRDQEEATNLPLSER
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028597747.1  DVPL~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028597747.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028597747.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
XP_028597747.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028597745.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028597745.1/" target="_blank">XP_028597745.1</a></a></td><td>XP_028597745</td><td>PDGF-B</td><td>snake venom vascular endothelial growth factor toxin HF-like isoform X2 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028597745.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028597745.1');">alignment</a><div id="align_XP_028597745.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  ~~~~~~~~~~~~~MPSSADSGAGRAWTSLLQHRSSSRSRTKRMPVPGYLV
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX
XP_028597745.1  AIALLCAQPVLTSEQPAGVVPFEEVWSRSYCRARETLVDILSEFPHEAEH
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT

           151                                            200
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX
XP_028597745.1  IFKPPCVPLWRCAGCCGDESLECIAVETRTIDLQVVRVSPILG.TTQQEE
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT

           201                                            250
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  MKFVEHTRCSCRPRRKRLKGERSHRIGAKRRPREPSLTPFTPLCKLPGAG
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  RPEAQHRDQEEATNLPLSERDVPL~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597745.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS

                401             419
VEGFC_signature ~~~~~~~~~~~~~~~~~~~
XP_028597745.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS



</pre></div></td></tr>
<tr><td><a id=XP_028597744.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028597744.1/" target="_blank">XP_028597744.1</a></a></td><td>XP_028597744</td><td>PDGF-B</td><td>snake venom vascular endothelial growth factor toxin HF-like isoform X1 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028597744.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028597744.1');">alignment</a><div id="align_XP_028597744.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  ~~~~~~~~~~~~~MPSSADSGAGRAWTSLLQHRSSSRSRTKRMPVPGYLV
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX
XP_028597744.1  AIALLCAQPVLTSEQPAGVVPFEEVWSRSYCRARETLVDILSEFPHEAEH
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT

           151                                            200
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX
XP_028597744.1  IFKPPCVPLWRCAGCCGDESLECIAVETRTIDLQVVRVSPILG.TTQQEE
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT

           201                                            250
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  MKFVEHTRCSCRPRRKRLKGERSHRIGAKRRPREPSLTPFTPLSCKLPGA
VEGF-C          ISFANHTSCRCMSKLDVYR..QVHSIIRRSLPATLPQCQAANKTCPTNYM

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  GRPEAQHRDQEEATNLPLSERDVPL~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          WNNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAG

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          LRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCP

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPG

                401               421
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~
XP_028597744.1  ~~~~~~~~~~~~~~~~~~~~~
VEGF-C          FSYSEEVCRCVPSYWKRPQMS



</pre></div></td></tr>
<tr><td><a id=XP_028575451.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028575451.1/" target="_blank">XP_028575451.1</a></a></td><td>XP_028575451</td><td>PDGF-B</td><td>placenta growth factor [Podarcis muralis]</td><td>PlGF-1</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028575451.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028575451.1');">alignment</a><div id="align_XP_028575451.1" style="display:none"><pre>


           1                                               50
XP_028575451.1  MCCRNRGGIASRSESELETGAREKRREPAPAMQLFGGFLVALALRVVGAQ
PlGF-1          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MPVMRLFPCFLQLLAGLALPAV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028575451.1  RQDVLLQN..STTEAAVVSFHDVWNRSSCQPLEKLVDVVSEYPSEVEHVF
PlGF-1          PPQWALSAGNGSSEVEVVPFQEVWGRSYCRALERLVDVVSEYPSEVEHMF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXX

           101                                            150
XP_028575451.1  SPSCVFLQRCSGCCGDERLQCVPTEVANVTMQLLKVNSEEQAPYVELSFV
PlGF-1          SPSCVSLLRCTGCCGDENLHCVPVETANVTMQLLKIRSGDRPSYVELTFS
VEGFC_signature XPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

                151                                          198
XP_028575451.1  EHRKCVCRPRRDAVKSGRRRRPKGRSKRKRAKQRPKD~~~~~~~~~~~
PlGF-1          QHVRCECRPLREKMKP.ERRRPKGRGKRRREKQRPTDCHLCGDAVPRR
VEGFC_signature XXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03699.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03699.1/" target="_blank">ADB03699.1</a></a></td><td>ADB03699</td><td>PDGF-C</td><td>vascular endothelial growth factor A, partial [Carinascincus metallicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03699.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03699.1');">alignment</a><div id="align_ADB03699.1" style="display:none"><pre>


           1                                               50
ADB03699.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03699.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWIHWGLAAWLY
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03699.1      FHHAKLSQAAPAQESGYVIPFLEVYER.SRCRPIETMVDISHEYPEEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03699.1      IFKPSCVSLMRCGGCCSDEALECVPLELQNVTMEVMRIKIHQSQH..ITR
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03699.1      MTFQQHKKCECRPKKETRHIKEKKSKRGKGKGQKRKRKRGRYKP....QT
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03699.1      FHCEPCSEKRKHLFVQDAQTCKCSCKYTDARCKLRQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSDAG.DDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03699.1      R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03699.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
ADB03699.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03700.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03700.1/" target="_blank">ADB03700.1</a></a></td><td>ADB03700</td><td>PDGF-B</td><td>vascular endothelial growth factor A, partial [Pseudemoia entrecasteauxii]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03700.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03700.1');">alignment</a><div id="align_ADB03700.1" style="display:none"><pre>


           1                                               50
ADB03700.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03700.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWIHWGLAALLY
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03700.1      FHHAKLSQAAPAQESGDVIPFMEVFAR.SRCKPIETMVDIFQEYPEEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03700.1      IFKPSCVPLMRCAGCCNDEALECMPLEVENVTMEVMRIKFHQSQHVSRMS
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03700.1      FQQHKTCECRPKKETRAIKENHCEPCSEKRKHLFVQDAQTCKCSCKYTDA
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03700.1      RCKLRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03700.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03700.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADB03700.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03693.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03693.1/" target="_blank">ADB03693.1</a></a></td><td>ADB03693</td><td>PDGF-C</td><td>vascular endothelial growth factor A, partial [Eulamprus tympanum]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03693.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03693.1');">alignment</a><div id="align_ADB03693.1" style="display:none"><pre>


           1                                               50
ADB03693.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03693.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHN
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03693.1      AKLSQAAPAQGDEEKQPSEVISFLEVFRRSGCRSIETMVDIFQEYPDEVE
VEGF-C          SRTEETIKFAAAHYN.TEILKSIDNEWRKTQCMPREVCIDVGKEFGVATN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXX

           151                                            200
ADB03693.1      YIFKPSCVPLLRCAGCCNDEALECVPTEVHNVTMEILRFKYHQGQHISQM
VEGF-C          TFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPV
VEGFC_signature XXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03693.1      SFQQHKACECRPKKETRVIKELHCEPCSEKRKHLFVQDAQTCKCSCKFTD
VEGF-C          TISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03693.1      ARCKLRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03693.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03693.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
ADB03693.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015284721.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015284721.1/" target="_blank">XP_015284721.1</a></a></td><td>XP_015284721</td><td>PDGF-B</td><td>PREDICTED: vascular endothelial growth factor A isoform X3 [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015284721.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015284721.1');">alignment</a><div id="align_XP_015284721.1" style="display:none"><pre>


           1                                               50
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015284721.1  LSQAAPAQGDGKKHHSEVIPFVD.VYNRSVCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015284721.1  FFKPSCVPLMRCAGCCNDEALECVALEVHNVTMEIMRIKPYQSQHIN..L
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015284721.1  MSFQHHSKCGCRPKKETRTRQEKCEKPRR~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
XP_015284721.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03701.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03701.1/" target="_blank">ADB03701.1</a></a></td><td>ADB03701</td><td>PDGF-B</td><td>vascular endothelial growth factor A, partial [Lampropholis guichenoti]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03701.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03701.1');">alignment</a><div id="align_ADB03701.1" style="display:none"><pre>


           1                                               50
ADB03701.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03701.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWIHWGLAALLY
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03701.1      FHHAKLSQAAPAQESGDVIPFMEVFAR.SRCKPIETMVDIFQEYPEEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03701.1      IFKPSCVPLMRCARCCNDEALECMP..LEVENVTMEVMRIKFHQGQHVSR
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03701.1      MSFQQHKTCECRPKKETRAIKENHCEPCSEKRKHLFVQDAQTCKCSCKYT
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03701.1      DARCKLRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03701.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03701.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADB03701.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015276119.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015276119.1/" target="_blank">XP_015276119.1</a></a></td><td>XP_015276119</td><td>PDGF-B</td><td>PREDICTED: placenta growth factor [Gekko japonicus]</td><td>PlGF-1</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015276119.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015276119.1');">alignment</a><div id="align_XP_015276119.1" style="display:none"><pre>


           1                                               50
XP_015276119.1  ~~~~~~MPTLGGFLVALALQTVGAKQEDFFHRNSTPEVGVLSFYDVWNRS
PlGF-1          MPVMRLFPCFLQLLAGLALPAVPP.QWALSAGNGSSEVEVVPFQEVWGRS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015276119.1  SCQPMEKLVNIVSEYPCEVEHMFSPSCVSLHRCSGCCGDETLHCVPTEMT
PlGF-1          YCRALERLVDVVSEYPSEVEHMFSPSCVSLLRCTGCCGDENLHCVPVETA
VEGFC_signature ~CXXXXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXX

           101                                            150
XP_015276119.1  NVTMQLLRMKSGEQASYVEMSFAEHKKCLCRLRQEVPKPGRRRRPKGKGR
PlGF-1          NVTMQLLKIRSGDRPSYVELTFSQHVRCECRPLREKMKP.ERRRPKGRGK
VEGFC_signature XXXXXXXXXXXXXXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~

                151               171
XP_015276119.1  GSKMKRARHRLKD~~~~~~~~
PlGF-1          RRREKQRPTDCHLCGDAVPRR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015284720.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015284720.1/" target="_blank">XP_015284720.1</a></a></td><td>XP_015284720</td><td>PDGF-A</td><td>PREDICTED: vascular endothelial growth factor A isoform X2 [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015284720.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015284720.1');">alignment</a><div id="align_XP_015284720.1" style="display:none"><pre>


           1                                               50
XP_015284720.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015284720.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015284720.1  LSQAAPAQGDGKKHHSEVIPFVD.VYNRSVCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015284720.1  FFKPSCVPLMRCAGCCNDEALECVALEVHNVTMEIMRIKPYQSQHIN..L
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015284720.1  MSFQHHSKCGCRPKKETRTRQENHCEPCSERRKHLFVQDPQTCKCSCKFT
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015284720.1  DSRCKSRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015284720.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015284720.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
XP_015284720.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028580181.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028580181.1/" target="_blank">XP_028580181.1</a></a></td><td>XP_028580181</td><td>PDGF-C</td><td>vascular endothelial growth factor A isoform X5 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028580181.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028580181.1');">alignment</a><div id="align_XP_028580181.1" style="display:none"><pre>


           1                                               50
XP_028580181.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028580181.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWMHWGLAALLYFQN
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028580181.1  AKFSQAAPAQGDGEKQQSEVISFLVVFERSACRTIETMVDIFQEYPDEVE
VEGF-C          SRTEETIKFAAAHYN.TEILKSIDNEWRKTQCMPREVCIDVGKEFGVATN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXX

           151                                            200
XP_028580181.1  YIFKPSCVPLMRCAGCCNDEALECVPTSTYNVTMEILRIKHFQGQHLSRM
VEGF-C          TFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPV
VEGFC_signature XXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028580181.1  SFQQHRECECRERPKKEVKPRQEKKSKRGKGKGQKRKRKRGRYKPQNLCE
VEGF-C          TISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028580181.1  KPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028580181.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028580181.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_028580181.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028580180.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028580180.1/" target="_blank">XP_028580180.1</a></a></td><td>XP_028580180</td><td>PDGF-C</td><td>vascular endothelial growth factor A isoform X4 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028580180.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028580180.1');">alignment</a><div id="align_XP_028580180.1" style="display:none"><pre>


           1                                               50
XP_028580180.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028580180.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWMHWGLAALLYFQN
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028580180.1  AKFSQAAPAQGDGEKQQSEVISFLVVFERSACRTIETMVDIFQEYPDEVE
VEGF-C          SRTEETIKFAAAHYN.TEILKSIDNEWRKTQCMPREVCIDVGKEFGVATN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXX

           151                                            200
XP_028580180.1  YIFKPSCVPLMRCAGCCNDEALECVPTSTYNVTMEILRIKHFQGQHLSRM
VEGF-C          TFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPV
VEGFC_signature XXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028580180.1  SFQQHRECECRERPKKEVKPRQENHCEPCSERRKHLFKQDPQTCKCSCKF
VEGF-C          TISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028580180.1  TDSRCKSRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028580180.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028580180.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_028580180.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF49643.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF49643.1/" target="_blank">GCF49643.1</a></a></td><td>GCF49643</td><td>PDGF-C</td><td>hypothetical protein parPi_0010403, partial [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF49643.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF49643.1');">alignment</a><div id="align_GCF49643.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      SWTGSREPAADQSEPEPEPASRSAACGRFGPEPASGAPEAGLAPGGCLWE
VEGF-C          ~~~~~~~~MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAE

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      PRVPRQGAPETRRPGRWCGCRWAGHARGLRSVLGWKSERAAEPAMPTLGA
VEGF-C          PDAGEATAYASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQH

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXX
GCF49643.1      FLVALALQTVGAKEDLFPRNSTPEEGVLSFYDVWNRSSCQPMEKLVNIVS
VEGF-C          NREQANLNSRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGK

           151                                            200
VEGFC_signature XXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXX
GCF49643.1      EYPCEVEHMFSPSCVSLHRCSGCCGDETLQCVPTEMTNVTMQLLRMKSGN
VEGF-C          EFGVATNTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPL

           201                                            250
VEGFC_signature XXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      QASYVEMSFAEHKKCLCRLRQEILKPGRRRRPKGRGRGSKMKRARHRLKD
VEGF-C          SQGPKPVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKT

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          CPTNYMWNNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQ

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          CVCRAGLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCV

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          CKRTCPRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQ

                401                     427
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF49643.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          KACEPGFSYSEEVCRCVPSYWKRPQMS



</pre></div></td></tr>
<tr><td><a id=XP_015284719.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015284719.1/" target="_blank">XP_015284719.1</a></a></td><td>XP_015284719</td><td>PDGF-B</td><td>PREDICTED: vascular endothelial growth factor A isoform X1 [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015284719.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015284719.1');">alignment</a><div id="align_XP_015284719.1" style="display:none"><pre>


           1                                               50
XP_015284719.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015284719.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015284719.1  LSQAAPAQGDGKKHHSEVIPFVD.VYNRSVCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015284719.1  FFKPSCVPLMRCAGCCNDEALECVALEVHNVTMEIMRIKPYQSQHIN..L
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015284719.1  MSFQHHSKCGCRPKKETRTRQEKKSKRGRGKGQKRKRKRGRYKPQ....N
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015284719.1  FHCEPCSERRKHLFVQDPQTCKCSCKFTDSRCKSRQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSD.AGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015284719.1  R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015284719.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_015284719.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32848.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32848.1/" target="_blank">AJR32848.1</a></a></td><td>AJR32848</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Anomalopus leuckartii]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32848.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32848.1');">alignment</a><div id="align_AJR32848.1" style="display:none"><pre>


           1                                               50
AJR32848.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32848.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32848.1      LSQAAPAQGDEEKQPSEVITFIEVFSR.SGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32848.1      IFKPSCVPLLRCAGCCNDEALECVPTEVHNVTMEILRIKY..HQGQHLSR
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32848.1      MSFQQHKTCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32848.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32848.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32848.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32848.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028580177.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028580177.1/" target="_blank">XP_028580177.1</a></a></td><td>XP_028580177</td><td>PDGF-C</td><td>vascular endothelial growth factor A isoform X2 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028580177.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028580177.1');">alignment</a><div id="align_XP_028580177.1" style="display:none"><pre>


           1                                               50
XP_028580177.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028580177.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWMHWGLAALLYFQNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028580177.1  FSQAAPAQGDGEKQQSEVISFLVVFER.SACRTIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_028580177.1  IFKPSCVPLMRCAGCCNDEALECVPTST..YNVTMEILRIKHFQGQHLSR
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028580177.1  MSFQQHRECECR.....ERPKKEVKPRQEKKSK...RGKGKGQKRKRKRG
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028580177.1  RHCEPCSERRKHLFKQDPQTCKCSCKFTDSRCKSRQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSD.AGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028580177.1  R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028580177.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_028580177.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03692.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03692.1/" target="_blank">ADB03692.1</a></a></td><td>ADB03692</td><td>PDGF-C</td><td>vascular endothelial growth factor A, partial [Eulamprus tympanum]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03692.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03692.1');">alignment</a><div id="align_ADB03692.1" style="display:none"><pre>


           1                                               50
ADB03692.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03692.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNA
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03692.1      KLSQAAPAQGDEEKQPSEVISFLEVFRRSGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03692.1      IFKPSCVPLLRCAGCCNDEALECVP..TEVHNVTMEILRFKYHQGQHISQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03692.1      MSFQQHKACECRPKKETRVIKEKKSKRGKGKGQKRKRKKSR....YKHQT
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03692.1      LHCEPCSEKRKHLFVQDAQTCKCSCKFTDARCKLRQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSDAG.DDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03692.1      R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03692.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
ADB03692.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028580176.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028580176.1/" target="_blank">XP_028580176.1</a></a></td><td>XP_028580176</td><td>PDGF-C</td><td>vascular endothelial growth factor A isoform X1 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028580176.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028580176.1');">alignment</a><div id="align_XP_028580176.1" style="display:none"><pre>


           1                                               50
XP_028580176.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028580176.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWMHWGLAALLYFQNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028580176.1  FSQAAPAQGDGEKQQSEVISFLVVFER.SACRTIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_028580176.1  IFKPSCVPLMRCAGCCNDEALECVPTSTYNVTMEILRIKHFQGQHLSRMS
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028580176.1  FQQHRECECRERPKKEVKPRQEKKSKRGKGKGQKRKRKRGRYKPQ....N
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028580176.1  FHCEPCSERRKHLFKQDPQTCKCSCKFTDSRCKSRQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSD.AGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028580176.1  R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028580176.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_028580176.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015284783.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015284783.1/" target="_blank">XP_015284783.1</a></a></td><td>XP_015284783</td><td>PDGF-B</td><td>PREDICTED: LOW QUALITY PROTEIN: snake venom vascular endothelial growth factor toxin HF-like [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015284783.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015284783.1');">alignment</a><div id="align_XP_015284783.1" style="display:none"><pre>


           1                                               50
XP_015284783.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015284783.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MTKLGNGNEDLE
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015284783.1  LSLGRPGGLEEACGCLEALP.FEVVWDLSFCASRVTLVDVLSEFPHDVEX
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015284783.1  IYKPLCVPLSRCNGCCNDERLECAPVETRPLDLQVMRISV.LQRTAEQVV
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015284783.1  LTFMEHMRCTCRRREMRWKDER..HRRTGAKRRPRKPSPAPLAPPPGWFY
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015284783.1  HHPNSYHTVH~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015284783.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015284783.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
XP_015284783.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32850.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32850.1/" target="_blank">AJR32850.1</a></a></td><td>AJR32850</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Ctenotus taeniolatus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32850.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32850.1');">alignment</a><div id="align_AJR32850.1" style="display:none"><pre>


           1                                               50
AJR32850.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32850.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32850.1      LSQAAPAHGDEEKQPSEVIPFLEVFGR.SGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32850.1      IFKPSCVPLLRCAGCCNDEALECVP..TEVHNVTMEIMRIKYHQGQHISQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32850.1      MSFQQHKTCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32850.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32850.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32850.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32850.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03694.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03694.1/" target="_blank">ADB03694.1</a></a></td><td>ADB03694</td><td>PDGF-A</td><td>vascular endothelial growth factor A, partial [Ctenotus taeniolatus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03694.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03694.1');">alignment</a><div id="align_ADB03694.1" style="display:none"><pre>


           1                                               50
ADB03694.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03694.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLY
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03694.1      FHNAKLSQAAPAHGDEEKQPSEVIPFLEVFGRSGCRSIETMVDIFQEYPD
VEGF-C          SRTEETIKFAAAHYNTEILKS....IDNEWRKTQCMPREVCIDVGKEFGV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXX

           151                                            200
ADB03694.1      EVEYIFKPSCVPLLRCAGCCNDEALECVPTEVHNVTMEIMRIKYHQGQHI
VEGF-C          ATNTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGP
VEGFC_signature XXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03694.1      SQMSFQQHKTCECRPKKETRVIKENHCEPCSEKRKHLFVQDAQTCKCSCK
VEGF-C          KPVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTN
VEGFC_signature XXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03694.1      FTDARCKLRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YMWNNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03694.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          AGLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03694.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          CPRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                 423
ADB03694.1      ~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PGFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32847.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32847.1/" target="_blank">AJR32847.1</a></a></td><td>AJR32847</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Calyptotis ruficauda]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32847.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32847.1');">alignment</a><div id="align_AJR32847.1" style="display:none"><pre>


           1                                               50
AJR32847.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32847.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32847.1      LSQAAPAQGDEEKQPSEVIQFLEVFDR.SRCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32847.1      IFKPSCVPLLRCAGCCNDEALECVP..IEVHNVTMEIMRIKYHQGQHINQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32847.1      MSFQQHKTCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32847.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32847.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32847.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32847.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028580178.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028580178.1/" target="_blank">XP_028580178.1</a></a></td><td>XP_028580178</td><td>PDGF-C</td><td>vascular endothelial growth factor A isoform X3 [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028580178.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028580178.1');">alignment</a><div id="align_XP_028580178.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
XP_028580178.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
XP_028580178.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWMHWGLAALLYFQNA

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
XP_028580178.1  KFSQAAPAQGDGEKQQSEVISFLVVFERSACRTIETMVDIFQEYPDEVEY

           151                                            200
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
XP_028580178.1  IFKPSCVPLMRCAGCCNDEALECVPTSTYNVTMEILRIKHFQGQHLSRMS

           201                                            250
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
XP_028580178.1  FQQHRECECRERPKKEVKPRQEKKSKRGKGKGQKRKRKRVTVNLAQKGES

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
XP_028580178.1  TCSNKIPRPVNVPANSQTHVASRGSLS~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
XP_028580178.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
XP_028580178.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
VEGFC_signature ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
XP_028580178.1  ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03695.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03695.1/" target="_blank">ADB03695.1</a></a></td><td>ADB03695</td><td>PDGF-B</td><td>vascular endothelial growth factor A, partial [Ctenotus taeniolatus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03695.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03695.1');">alignment</a><div id="align_ADB03695.1" style="display:none"><pre>


           1                                               50
ADB03695.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03695.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03695.1      LSQAAPAHGDEEKQPSEVIPFLEVFGR.SGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03695.1      IFKPSCVPLLRCAGCCNDEALECVP..TEVHNVTMEIMRIKYHQGQHISQ
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03695.1      MSFQQHKTCECRPKKETRVIKEKKSKR..GKGKGQKRKRKKSRYKHQTFH
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03695.1      CEPCSEKRKHLFVQDAQTCKCSCK.FTDARCKLRQLELNERTCRCEKPRR
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03695.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03695.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADB03695.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF54425.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF54425.1/" target="_blank">GCF54425.1</a></a></td><td>GCF54425</td><td>PDGF-B</td><td>hypothetical protein parPi_0016499 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF54425.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF54425.1');">alignment</a><div id="align_GCF54425.1" style="display:none"><pre>


           1                                               50
GCF54425.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF54425.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLAWLRWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF54425.1      LSQAAPTQGDGKKHHSEVIQFLD.VYNRSMCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
GCF54425.1      IFMPSCVPLLRCAGCCNDEALECVALEVHNVTMEIMRIKPLVSQHIN..L
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
GCF54425.1      MSFQHHSKCGCRPKKETRTKLEKKSKRGRGKGQKRKRKRGRYKPQ....N
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF54425.1      FHCEPCSERRKHLFVQDPQTCKCSCKFTDSRCKSKQLELNERTCRCEKPR
VEGF-C          NHICRCLAQEDFMFSSD.AGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
GCF54425.1      R~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
GCF54425.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
GCF54425.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03698.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03698.1/" target="_blank">ADB03698.1</a></a></td><td>ADB03698</td><td>PDGF-C</td><td>vascular endothelial growth factor A, partial [Saiphos equalis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03698.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03698.1');">alignment</a><div id="align_ADB03698.1" style="display:none"><pre>


           1                                               50
ADB03698.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03698.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03698.1      LSQAAPAQGEEEKQPSEVIPFMEVFGR.SGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03698.1      IFKPSCVPLLRCAGCCNDEALECVP..TEVHNVTMEIMRLKYHQGQHLSW
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03698.1      MSFQQHKTCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03698.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03698.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03698.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADB03698.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03697.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03697.1/" target="_blank">ADB03697.1</a></a></td><td>ADB03697</td><td>PDGF-B</td><td>vascular endothelial growth factor A, partial [Saiphos equalis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03697.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03697.1');">alignment</a><div id="align_ADB03697.1" style="display:none"><pre>


           1                                               50
ADB03697.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03697.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHN
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03697.1      AKLSQAAPAQGEEEKQPSEVIPFMEVFGRSGCRSIETMVDIFQEYPDEVE
VEGF-C          SRTEETIKFAAAHYN.TEILKSIDNEWRKTQCMPREVCIDVGKEFGVATN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXX

           151                                            200
ADB03697.1      YIFKPSCVPLLRCAGCCNDEALECVPTEVHNVTMEIMRLKYHQGQHLSWM
VEGF-C          TFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPV
VEGFC_signature XXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03697.1      SFQQHKTCECRPKKETRVIKENHCEPCSEKRKHLFVQDAQTCKCSCKFTD
VEGF-C          TISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03697.1      ARCKLRQLELNERTCRCEKPRR~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03697.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03697.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
ADB03697.1      ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=ADB03696.1><a href="https://www.ncbi.nlm.nih.gov/protein/ADB03696.1/" target="_blank">ADB03696.1</a></a></td><td>ADB03696</td><td>PDGF-B</td><td>vascular endothelial growth factor A, partial [Saiphos equalis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=ADB03696.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_ADB03696.1');">alignment</a><div id="align_ADB03696.1" style="display:none"><pre>


           1                                               50
ADB03696.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
ADB03696.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
ADB03696.1      LSQAAPAQGEEEKQPSEVIPFMEVFGR.SGCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
ADB03696.1      IFKPSCVPLLRCAGCCNDEALECVP..TEVHNVTMEIMRLKYHQGQHLSW
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
ADB03696.1      MSFQQHKTCECRPKKETRVIKEKKSKR..GKGKGQKRKRKKSRYKHQTFH
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
ADB03696.1      CEPCSEKRKHLFVQDAQTCKCSCK.FTDARCKLRQLELNERTCRCEKPRR
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
ADB03696.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
ADB03696.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
ADB03696.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32851.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32851.1/" target="_blank">AJR32851.1</a></a></td><td>AJR32851</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Lerista bougainvillii]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32851.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32851.1');">alignment</a><div id="align_AJR32851.1" style="display:none"><pre>


           1                                               50
AJR32851.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32851.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32851.1      LSQAAPAQGDEEKKPSEVIP.FYEVFDRSSCRSIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32851.1      IFKPSCVPLLRCAGCCNDEALECVPIEVHNVTMEIMRIKYHQSQHISQMS
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32851.1      FQQHKKCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32851.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32851.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32851.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32851.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32852.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32852.1/" target="_blank">AJR32852.1</a></a></td><td>AJR32852</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Lerista bougainvillii]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32852.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32852.1');">alignment</a><div id="align_AJR32852.1" style="display:none"><pre>


           1                                               50
AJR32852.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32852.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFXLTWLHWGLXALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32852.1      LSQAAPAQGDEEKKPSEVIP.FYEVFDRSSCRAIETMVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32852.1      IFKPSCVPLLRCAGCCNDEALECVPIEVHNVTMEIMRIKYHQSQHIS..Q
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32852.1      MSFQQHKKCECTCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32852.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32852.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32852.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32852.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=AJR32849.1><a href="https://www.ncbi.nlm.nih.gov/protein/AJR32849.1/" target="_blank">AJR32849.1</a></a></td><td>AJR32849</td><td>PDGF-C</td><td>VEGF111 splice variant, partial [Hemiergis peronii]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=AJR32849.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_AJR32849.1');">alignment</a><div id="align_AJR32849.1" style="display:none"><pre>


           1                                               50
AJR32849.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
AJR32849.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNFLLTWLHWGLAALLYFHNAK
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
AJR32849.1      LSQAAPAQGNEEKEPSEVIPFFEVFSR.SGCRSIETLVDIFQEYPDEVEY
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
AJR32849.1      IFKPSCVPLLRCAGCCNDEALECVPIEVHNVTMEIMRIKYHQSQHISQMS
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
AJR32849.1      FQQHKTCECRCEKP~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
AJR32849.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
AJR32849.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
AJR32849.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
AJR32849.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015272314.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015272314.1/" target="_blank">XP_015272314.1</a></a></td><td>XP_015272314</td><td>PDGF-B</td><td>PREDICTED: vascular endothelial growth factor B [Gekko japonicus]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015272314.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015272314.1');">alignment</a><div id="align_XP_015272314.1" style="display:none"><pre>


           1                                               50
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MNAPCNALQLLLATTLQILYC
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015272314.1  AATVMRPEADATHPSGEVIRWMD.VYNRSFCQPKETLVPVSAEHPGEVEH
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
XP_015272314.1  ILVPSCVPLRRCVGCCSDEELQCVPVQMHVVIMEVMKTRFPRSHLG...Q
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_015272314.1  MSFVEHSACECR~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
XP_015272314.1  ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF46055.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF46055.1/" target="_blank">GCF46055.1</a></a></td><td>GCF46055</td><td>PDGF-B</td><td>hypothetical protein parPi_0005736 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF46055.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF46055.1');">alignment</a><div id="align_GCF46055.1" style="display:none"><pre>


           1                                               50
GCF46055.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF46055.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~MPAPCGALQLLLATTLQLLSCA
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF46055.1      ADTMMSPEADATHPSGEVIR.WMEVYNRSFCQPKETLVPVSAEHPGEVEH
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX

           151                                            200
GCF46055.1      ILVPSCVPLRRCVGCCSDEELQCVPVQMHVVIMEVMKTRFPHSHLG...Q
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
GCF46055.1      MSFVEHSACECRPKKLIADKTERLP..CPPCLDRRKQPDPETCKCHCRRR
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF46055.1      PQRCRNLGLELNEHSCRCEKLRR~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
GCF46055.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
GCF46055.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401             419
GCF46055.1      ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF40753.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF40753.1/" target="_blank">GCF40753.1</a></a></td><td>GCF40753</td><td>PDGF-D</td><td>hypothetical protein parPi_0001015 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF40753.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF40753.1');">alignment</a><div id="align_GCF40753.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
GCF40753.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
GCF40753.1      ~~~~~~~~~~~~~~~~~~~MLLSGFLWLTFALVSQRQGARAESNLSSKFQ

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXX
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT
GCF40753.1      FSGGKEQNGVQEPHHEKIITVSANGSIHSPKFPFTYPRNTVLVWRLVAIQ

           151                                            200
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXX
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPVT
GCF40753.1      ENVLIQLTFDERFGLEDPEDDICKYDFVEIEEPSDGSILGRWCGSTAVPG

           201                                            250
VEGFC_signature XXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          ISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMWN
GCF40753.1      KQISKGNQIRIRFVSDEYFPSEPGFCIHYTLLTPHQTESASPTVLPPSAF

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGLR
GCF40753.1      SLDLLNNAVAGFSTVEELIRYLEPDRWQLDLEDLYKPAWQLLGKAYIHGR

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPRN
GCF40753.1      KSRVVDLNLLKEEVRMYSCTPRNFSVSLREELKRTDTIFWPLCLLVKRCG

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          QPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGFS
GCF40753.1      GNCACCQHSCSECQCVPTKVTKKYHEVLQLKPRSGVRGLHKSLTDVPLEH

                401             419
VEGFC_signature ~~~~~~~~~~~~~~~~~~~
VEGF-C          YSEEVCRCVPSYWKRPQMS
GCF40753.1      HEECECICKGNAEG~~~~~



</pre></div></td></tr>
<tr><td><a id=NP_001310426.1><a href="https://www.ncbi.nlm.nih.gov/protein/NP_001310426.1/" target="_blank">NP_001310426.1</a></a></td><td>NP_001310426</td><td>PDGF-D</td><td>platelet-derived growth factor C precursor [Gekko japonicus]</td><td>PDGF-C</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=NP_001310426.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_NP_001310426.1');">alignment</a><div id="align_NP_001310426.1" style="display:none"><pre>


           1                                               50
NP_001310426.1  MLLFGFLLLTFALVSQRQGAEAESNLSSKFQFSSAKEQNGVQEPQHEKII
PDGF-C          MSLFGLLLLTSALAGQRQGTQAESNLSSKFQFSSNKEQNGVQDPQHERII
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
NP_001310426.1  TVSANGSIHSPKFPYTYPRNTVLVWRLVAIEENVLIQLTFDERFGLEDPE
PDGF-C          TVSTNGSIHSPRFPHTYPRNTVLVWRLVAVEENVWIQLTFDERFGLEDPE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
NP_001310426.1  DDICKYDFVEVEEPSDGSILGRWCGSTAVPGKQISKGNQIRIRFVSDEYF
PDGF-C          DDICKYDFVEVEEPSDGTILGRWCGSGTVPGKQISKGNQIRIRFVSDEYF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
NP_001310426.1  PSEPGFCIHYTLLTPHQTESASPTVLPPSAFSLDLLNNAVAGFSTVEELI
PDGF-C          PSEPGFCIHYNIVMPQFTEAVSPSVLPPSALPLDLLNNAITAFSTLEDLI
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
NP_001310426.1  RYLEPDRWQLDLEDLYKPAWQLLGKAYIHGRKSRVVDLNLLKEEVRMYSC
PDGF-C          RYLEPERWQLDLEDLYRPTWQLLGKAFVFGRKSRVVDLNLLTEEVRLYSC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~C

           251                                            300
NP_001310426.1  TPRN.FSVSLREELKRTDTIFWPLCLLVKRCGGNCACCQHSCSECQCIPT
PDGF-C          TPRN.FSVSIREELKRTDTIFWPGCLLVKRCGGNCACCLHNCNECQCVPS
VEGFC_signature XXXXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXX

                301                                        346
NP_001310426.1  KVTKKYHEVLQLKPRSGVRGLHKSLTDVPLEHHEECECVCKGNAEG
PDGF-C          KVTKKYHEVLQLRPKTGVRGLHKSLTDVALEHHEECDCVCRGSTGG
VEGFC_signature XXXXXXXXXXXXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028566508.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028566508.1/" target="_blank">XP_028566508.1</a></a></td><td>XP_028566508</td><td>PDGF-B</td><td>vascular endothelial growth factor B [Podarcis muralis]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028566508.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028566508.1');">alignment</a><div id="align_XP_028566508.1" style="display:none"><pre>


           1                                               50
XP_028566508.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028566508.1  ~~~~~~~~~~~~~~~~~~~~~~~~~MNAPCGALQLLLATTLQLHYCAATT
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028566508.1  ILPHPTNEMEPVVEPVTEVVRFLEVFSRSACQAKETMVSLDKEYPDLVIS
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFG.VATN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXX.XXXX

           151                                            200
XP_028566508.1  HLIVPSCVALKRCVGCCADESAQCMPSRT..YTVTMEVLLLSKFQRSHLQ
VEGF-C          TFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPKPV
VEGFC_signature XXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXX

           201                                            250
XP_028566508.1  QLTFEEHATCECRPKEVYRLATHSPS..CAPCPDRKKRVNPVTCECVCKR
VEGF-C          TISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTNYMW
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028566508.1  DSVRCQQRGWELNKATCRCVKHRRRSGRKAKA~~~~~~~~~~~~~~~~~~
VEGF-C          NNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAGL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
XP_028566508.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCPR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
XP_028566508.1  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          NQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPGF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401              420
XP_028566508.1  ~~~~~~~~~~~~~~~~~~~~
VEGF-C          SYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028599432.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028599432.1/" target="_blank">XP_028599432.1</a></a></td><td>XP_028599432</td><td>PDGF-D</td><td>platelet-derived growth factor C [Podarcis muralis]</td><td>PDGF-C</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028599432.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028599432.1');">alignment</a><div id="align_XP_028599432.1" style="display:none"><pre>


           1                                               50
XP_028599432.1  MLLFGFFLLTSALVSPRQGAQGESNLSSKFQFSSAKEQNGVQEPQHEKII
PDGF-C          MSLFGLLLLTSALAGQRQGTQAESNLSSKFQFSSNKEQNGVQDPQHERII
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028599432.1  TVSANGSIHSPKFPYTYPRNTVLVWRLVAVEENVLIQLTFDERFGLEDPE
PDGF-C          TVSTNGSIHSPRFPHTYPRNTVLVWRLVAVEENVWIQLTFDERFGLEDPE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028599432.1  DDICKYDFVEVEEPSDGTILGRWCGSTTVPGKQISKGNQIRIRFVSDEYF
PDGF-C          DDICKYDFVEVEEPSDGTILGRWCGSGTVPGKQISKGNQIRIRFVSDEYF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
XP_028599432.1  PSEPGFCIHYTLLVP..TESASPTILPPSALPLELLNNAVAGFSTVEELI
PDGF-C          PSEPGFCIHYNIVMPQFTEAVSPSVLPPSALPLDLLNNAITAFSTLEDLI
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
XP_028599432.1  KYLEPDRWQLDLEDLYRPTWQLLGKAYIHGRKSRVVDLNLLKEEVRMYSC
PDGF-C          RYLEPERWQLDLEDLYRPTWQLLGKAFVFGRKSRVVDLNLLTEEVRLYSC
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~C

           251                                            300
XP_028599432.1  TPRNYSVSLREELKR.TDTIFWPLCLLVKRCGGNCGCCQHACSECQCVPT
PDGF-C          TPRNFSVSIREELKR.TDTIFWPGCLLVKRCGGNCACCLHNCNECQCVPS
VEGFC_signature XXXXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXX

                301                                        346
XP_028599432.1  KVTKKYHEVLQLKPRSGVKGLHKSLTDVPLDHHEECDCVCKGNTEG
PDGF-C          KVTKKYHEVLQLRPKTGVRGLHKSLTDVALEHHEECDCVCRGSTGG
VEGFC_signature XXXXXXXXXXXXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028601741.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028601741.1/" target="_blank">XP_028601741.1</a></a></td><td>XP_028601741</td><td>PDGF-D</td><td>platelet-derived growth factor subunit B isoform X3 [Podarcis muralis]</td><td>PDGF-B</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028601741.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028601741.1');">alignment</a><div id="align_XP_028601741.1" style="display:none"><pre>


           1                                               50
XP_028601741.1  MRAANGGTAKQASFPQRLLREEVKDRRMGDPIPEEIYELLSDSSLRSIRD
PDGF-B          ~~~~~~~MNRCWALFLSLCCYLRLVSAEGDPIPEELYEMLSDHSIRSFDD
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028601741.1  LQQVLRIDSVEEDNSTSYLNTTRIHPGQNPVSFLRGRRSLVGGEPVEPAV
PDGF-B          LQRLLHGDPGEEDGAELDLNMTRSHSGGELESLARGRRSLGSLTIAEPAM
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028601741.1  IAECKTRTEVFEISRTMVDPTNANFLVWPPCVEVQRCTGCCNTRNMQCRA
PDGF-B          IAECKTRTEVFEISRRLIDRTNANFLVWPPCVEVQRCSGCCNNRNVQCRP
VEGFC_signature ~~~CXX..XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXX

           151                                            200
XP_028601741.1  TRVRERHIQVNKIEIVRKKPVFNKAIVALEDHLECRCEAVSPRAFSWPNS
PDGF-B          TQVQLRPVQVRKIEIVRKKPIFKKATVTLEDHLACKCETVAAARPVTRSP
VEGFC_signature XXXXXXXXXXXXXXXX.XXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~

                201                                            250
XP_028601741.1  REHRAGGPPLSTRASLPARARAHRLLPQKRKHRKFKHVQEKKALKEIFPA
PDGF-B          GGSQEQRAKTPQTRVTIRTVRVR..RPPKGKHRKFKHTHDKTALKETLGA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028601742.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028601742.1/" target="_blank">XP_028601742.1</a></a></td><td>XP_028601742</td><td>PDGF-D</td><td>platelet-derived growth factor subunit B isoform X4 [Podarcis muralis]</td><td>PDGF-B</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028601742.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028601742.1');">alignment</a><div id="align_XP_028601742.1" style="display:none"><pre>


           1                                               50
XP_028601742.1  MCLELLGIGVGMNFGVLFVCFSYLPLVSSEGDPIPEEIYELLSDSSLRSI
PDGF-B          ~~~~~~~~~MNRCWALFLSLCCYLRLVSAEGDPIPEELYEMLSDHSIRSF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028601742.1  RDLQQVLRIDSVEEDNSTSYLNTTRIHPGQNPVSFLRGRRSLVGGEPVEP
PDGF-B          DDLQRLLHGDPGEEDGAELDLNMTRSHSGGELESLARGRRSLGSLTIAEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028601742.1  AVIAECKTRTEVFEISRTMVDPTNANFLVWPPCVEVQRCTGCCNTRNMQC
PDGF-B          AMIAECKTRTEVFEISRRLIDRTNANFLVWPPCVEVQRCSGCCNNRNVQC
VEGFC_signature ~~~~~CXX..XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXC

           151                                            200
XP_028601742.1  RATRVRERHIQVNKIEIVRKKPVFNKAIVALEDHLECRCEAVSPR...AF
PDGF-B          RPTQVQLRPVQVRKIEIVRKKPIFKKATVTLEDHLACKCETVAAARPVTR
VEGFC_signature XXXXXXXXXXXXXXXXXXXXX.XXXXXXXXXXXXXXCXC~~~~~~~~~~~

                201                                            250
XP_028601742.1  SWPNSREHRARARAHRLLP........QKRKHRKFKHVQEKKALKEIFPA
PDGF-B          SPGGSQEQRAKTPQTRVTIRTVRVRRPPKGKHRKFKHTHDKTALKETLGA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028601735.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028601735.1/" target="_blank">XP_028601735.1</a></a></td><td>XP_028601735</td><td>PDGF-D</td><td>platelet-derived growth factor subunit B isoform X1 [Podarcis muralis]</td><td>PDGF-B</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028601735.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028601735.1');">alignment</a><div id="align_XP_028601735.1" style="display:none"><pre>


           1                                               50
XP_028601735.1  MCLELLGIGVGMNFGVLFVCFSYLPLVSSEGDPIPEEIYELLSDSSLRSI
PDGF-B          ~~~~~~~~~MNRCWALFLSLCCYLRLVSAEGDPIPEELYEMLSDHSIRSF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028601735.1  RDLQQVLRIDSVEEDNSTSYLNTTRIHPGQNPVSFLRGRRSLVGGEPVEP
PDGF-B          DDLQRLLHGDPGEEDGAELDLNMTRSHSGGELESLARGRRSLGSLTIAEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028601735.1  AVIAECKTRTEVFEISRTMVDPTNANFLVWPPCVEVQRCTGCCNTRNMQC
PDGF-B          AMIAECKTRTEVFEISRRLIDRTNANFLVWPPCVEVQRCSGCCNNRNVQC
VEGFC_signature ~~~~~CXX..XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXC

           151                                            200
XP_028601735.1  RATRVRERHIQVNKIEIVRKKPVFNKAIVALEDHLECRCEAVSPRAFSWP
PDGF-B          RPTQVQLRPVQVRKIEIVRKKPIFKKATVTLEDHLACKCETVAAARPVTR
VEGFC_signature XXXXXXXXXXXXXXXXXX.XXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~

           201                                            250
XP_028601735.1  NSREHRAGGPPLSTRASLPARARAHRLLPQKRKHRKFKHVQEKKALKEIF
PDGF-B          SPGGSQEQRAKTPQTRVTIRTVRVR..RPPKGKHRKFKHTHDKTALKETL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                251 252
XP_028601735.1  PA
PDGF-B          GA
VEGFC_signature ~~



</pre></div></td></tr>
<tr><td><a id=XP_028601740.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028601740.1/" target="_blank">XP_028601740.1</a></a></td><td>XP_028601740</td><td>PDGF-D</td><td>platelet-derived growth factor subunit B isoform X2 [Podarcis muralis]</td><td>PDGF-B</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028601740.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028601740.1');">alignment</a><div id="align_XP_028601740.1" style="display:none"><pre>


           1                                               50
XP_028601740.1  MCLELLGIGVGMNFGVLFVCFSYLPLVSSEGDPIPEEIYELLSDSSLRSI
PDGF-B          ~~~~~~~~~MNRCWALFLSLCCYLRLVSAEGDPIPEELYEMLSDHSIRSF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028601740.1  RDLQQVLRIDSVEEDNSTSYLNTTRIHPGQNPVSFLRGRRSLVGGEPVEP
PDGF-B          DDLQRLLHGDPGEEDGAELDLNMTRSHSGGELESLARGRRSLGSLTIAEP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028601740.1  AVIAECKTRTEVFEISRTMVDPTNANFLVWPPCVEVQRCTGCCNTRNMQC
PDGF-B          AMIAECKTRTEVFEISRRLIDRTNANFLVWPPCVEVQRCSGCCNNRNVQC
VEGFC_signature ~~~~~CXX..XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXC

           151                                            200
XP_028601740.1  RATRVRERHIQVNKIEIVRKKPVFNKAIVALEDHLECRCEAVSPRAFSWP
PDGF-B          RPTQVQLRPVQVRKIEIVRKKPIFKKATVTLEDHLACKCETVAAARPVTR
VEGFC_signature XXXXXXXXXXXXXXXXXX.XXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~

           201                                            250
XP_028601740.1  NSREHRGGPPLSTRASLPARARAHRLLPQKRKHRKFKHVQEKKALKEIFP
PDGF-B          SPGGSQEQRAKTPQTRVTIRTVRVRRPP.KGKHRKFKHTHDKTALKETLG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                251 251
XP_028601740.1  A
PDGF-B          A
VEGFC_signature ~



</pre></div></td></tr>
<tr><td><a id=XP_015274676.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015274676.1/" target="_blank">XP_015274676.1</a></a></td><td>XP_015274676</td><td>PDGF-D</td><td>PREDICTED: platelet-derived growth factor subunit B [Gekko japonicus]</td><td>PDGF-B</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015274676.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015274676.1');">alignment</a><div id="align_XP_015274676.1" style="display:none"><pre>


           1                                               50
XP_015274676.1  MCLELLGIGIGMNFGVLFLSLWYLPLISTKGDPIPEEIYEMLSDSSVRSI
PDGF-B          ~~~~~~~~~MNRCWALFLSLCCYLRLVSAEGDPIPEELYEMLSDHSIRSF
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015274676.1  SDLRRVLQIDSVEEDGSNFYLNSTRIHPGRNHVPFFRGRRSIVSGGEEPI
PDGF-B          DDLQRLLHGDPGEEDGAELDLNMTRSHSGGELESLARGRRSLGS..LTIA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015274676.1  EAAVIAECKTRTEVFEISRSMVDPTNANFIVWPPCVEVQRCTGCCNTRNM
PDGF-B          EPAMIAECKTRTEVFEISRRLIDRTNANFLVWPPCVEVQRCSGCCNNRNV
VEGFC_signature ~~~~~~~CXX..XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXX

           151                                            200
XP_015274676.1  HCLPTRVRVRHIQVNKIGIVQKKPTFNKAIVALEDHLECRCEAVSPRPLS
PDGF-B          QCRPTQVQLRPVQVRKIEIVRKKPIFKKATVTLEDHLACKCETVAAARPV
VEGFC_signature XCXXXXXXXXXXXXXXXXXXXXX.XXXXXXXXXXXXXXCXC~~~~~~~~~

           201                                            250
XP_015274676.1  RAHSHEHRAGGPRLTTVGPLQARARAHRLMPQKRKHRKFKHVHDKKVLKD
PDGF-B          TRSPGGSQEQRAKTPQTRVTIRTVRVR..RPPKGKHRKFKHTHDKTALKE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251
XP_015274676.1  IFPA
PDGF-B          TLGA
VEGFC_signature ~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015285151.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015285151.1/" target="_blank">XP_015285151.1</a></a></td><td>XP_015285151</td><td>PDGF-D</td><td>PREDICTED: platelet-derived growth factor D isoform X3 [Gekko japonicus]</td><td>PDGF-D</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015285151.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015285151.1');">alignment</a><div id="align_XP_015285151.1" style="display:none"><pre>


           1                                               50
XP_015285151.1  MQLPAPFLLYVVAWANFCHRHGSAAQSGSIRALRASNLRRDESNHLTALY
PDGF-D          MHRLIFVYTLICANFCSCRDTSATPQSASIKALRNANLRRDESNHLTDLY
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015285151.1  RKEETINVRGNGHIHSPRFPNSYPRNLLLTWKLLSPENTRIQLVFDSQFG
PDGF-D          RRDETIQVKGNGYVQSPRFPNSYPRNLLLTWRLHSQENTRIQLVFDNQFG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015285151.1  LEEPENEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRITSRTNQLKIT
PDGF-D          LEEAENDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIKSRTNQIKIT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
XP_015285151.1  FKSDDYFVAKPGFKIYYSRVG.....................VSFHSPSV
PDGF-D          FKSDDYFVAKPGFKIYYSLLEDFQPAAASETNWESVTSSISGVSYNSPSV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
XP_015285151.1  TDPTLTAEALDQTIAGFDTVEELLRHFNPDTWQEDLETLYTDTAQYRGRN
PDGF-D          TDPTLIADALDKKIAEFDTVEDLLKYFNPESWQEDLENMYLDTPRYRGRS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015285151.1  FYERKSKVDLDRLNDDVKRYSCTPRNYSVNIREELRVTN.AVFFPRCLLV
PDGF-D          YHDRKSKVDLDRLNDDAKRYSCTPRNYSVNIREELKLAN.VVFFPRCLLV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXXXPXCVXX

           301                                            350
XP_015285151.1  QRCGGNCGCGFTHWKSCTCTPGKLVKKYHEVLRFIPEAGHIRRRGRTKNN
PDGF-D          QRCGGNCGCGTVNWRSCTCNSGKTVKKYHEVLQFEPG..HIKRRGRAKT.
VEGFC_signature XRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXC

                351                  374
XP_015285151.1  MALIDIQLEHHERCDCVCSSRPPR
PDGF-D          MALVDIQLDHHERCDCICSSRPPR
VEGFC_signature XC~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF41376.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF41376.1/" target="_blank">GCF41376.1</a></a></td><td>GCF41376</td><td>PDGF-D</td><td>hypothetical protein parPi_0001345 [Paroedura picta]</td><td>PDGF-D</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF41376.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF41376.1');">alignment</a><div id="align_GCF41376.1" style="display:none"><pre>


           1                                               50
GCF41376.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PDGF-D          MHRLIFVYTLICANFCSCRDTSATPQSASIKALRNANLRRDESNHLTDLY
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF41376.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PDGF-D          RRDETIQVKGNGYVQSPRFPNSYPRNLLLTWRLHSQENTRIQLVFDNQFG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF41376.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PDGF-D          LEEAENDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIKSRTNQIKIT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
GCF41376.1      ~~~~MGIVVHGHLEDHRLTEGDDLQPAASETNWESVTSSVSGVSFHSPSV
PDGF-D          FKSDDYFVAKPGFKIYYSLLEDFQPAAASETNWESVTSSISGVSYNSPSV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
GCF41376.1      TDPTLTAEALDQTIAGFDTVEELLRHFNPDTWQEDLETLYTDTAQYRGRN
PDGF-D          TDPTLIADALDKKIAEFDTVEDLLKYFNPESWQEDLENMYLDTPRYRGRS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF41376.1      FYEKKSKVDLDRLNDDVKRYSCTPRNYSVNIREELRVTN.AVFFPRCLLV
PDGF-D          YHDRKSKVDLDRLNDDAKRYSCTPRNYSVNIREELKLAN.VVFFPRCLLV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXXXPXCVXX

           301                                            350
GCF41376.1      QRCGGNCGCGFTHWKSCTCTPGKLVKKYHEVLRFIPEAGHIRRRGRTKNN
PDGF-D          QRCGGNCGCGTVNWRSCTCNSGKTVKKYHEVLQFEPG..HIKRRGRAKT.
VEGFC_signature XRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXC

                351                  374
GCF41376.1      MALIDIQLEHHERCDCVCSSRPPR
PDGF-D          MALVDIQLDHHERCDCICSSRPPR
VEGFC_signature XC~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015285149.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015285149.1/" target="_blank">XP_015285149.1</a></a></td><td>XP_015285149</td><td>PDGF-D</td><td>PREDICTED: platelet-derived growth factor D isoform X1 [Gekko japonicus]</td><td>PDGF-D</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015285149.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015285149.1');">alignment</a><div id="align_XP_015285149.1" style="display:none"><pre>


           1                                               50
XP_015285149.1  MQLPAPFLLYVVAWANFCHRHGSAAQSGSIRALRASNLRRDESNHLTALY
PDGF-D          MHRLIFVYTLICANFCSCRDTSATPQSASIKALRNANLRRDESNHLTDLY
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015285149.1  RKEETINVRGNGHIHSPRFPNSYPRNLLLTWKLLSPENTRIQLVFDSQFG
PDGF-D          RRDETIQVKGNGYVQSPRFPNSYPRNLLLTWRLHSQENTRIQLVFDNQFG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015285149.1  LEEPENEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRITSRTNQLKIT
PDGF-D          LEEAENDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIKSRTNQIKIT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
XP_015285149.1  FKSDDYFVAKPGFKIYYSRVDDLQPAAS.ETNWESVTSPVLGVSFHSPSV
PDGF-D          FKSDDYFVAKPGFKIYYSLLEDFQPAAASETNWESVTSSISGVSYNSPSV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
XP_015285149.1  TDPTLTAEALDQTIAGFDTVEELLRHFNPDTWQEDLETLYTDTAQYRGRN
PDGF-D          TDPTLIADALDKKIAEFDTVEDLLKYFNPESWQEDLENMYLDTPRYRGRS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015285149.1  FYERKSKVDLDRLNDDVKRYSCTPRNYSVNIREELRVTN.AVFFPRCLLV
PDGF-D          YHDRKSKVDLDRLNDDAKRYSCTPRNYSVNIREELKLAN.VVFFPRCLLV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXXXPXCVXX

           301                                            350
XP_015285149.1  QRCGGNCGCGFTHWKSCTCTPGKLVKKYHEVLRFIPEAGHIRRRGRTKNN
PDGF-D          QRCGGNCGCGTVNWRSCTCNSGKTVKKYHEVLQFEPG..HIKRRGRAKT.
VEGFC_signature XRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXC

                351                  374
XP_015285149.1  MALIDIQLEHHERCDCVCSSRPPR
PDGF-D          MALVDIQLDHHERCDCICSSRPPR
VEGFC_signature XC~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015285150.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015285150.1/" target="_blank">XP_015285150.1</a></a></td><td>XP_015285150</td><td>PDGF-D</td><td>PREDICTED: platelet-derived growth factor D isoform X2 [Gekko japonicus]</td><td>PDGF-D</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015285150.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015285150.1');">alignment</a><div id="align_XP_015285150.1" style="display:none"><pre>


           1                                               50
XP_015285150.1  MQLPAPFLLYVVAWANFCHRHGSAAQSGSIRALRASNLRRD......ALY
PDGF-D          MHRLIFVYTLICANFCSCRDTSATPQSASIKALRNANLRRDESNHLTDLY
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015285150.1  RKEETINVRGNGHIHSPRFPNSYPRNLLLTWKLLSPENTRIQLVFDSQFG
PDGF-D          RRDETIQVKGNGYVQSPRFPNSYPRNLLLTWRLHSQENTRIQLVFDNQFG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_015285150.1  LEEPENEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRITSRTNQLKIT
PDGF-D          LEEAENDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIKSRTNQIKIT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
XP_015285150.1  FKSDDYFVAKPGFKIYYSRVDDLQPAAS.ETNWESVTSPVLGVSFHSPSV
PDGF-D          FKSDDYFVAKPGFKIYYSLLEDFQPAAASETNWESVTSSISGVSYNSPSV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
XP_015285150.1  TDPTLTAEALDQTIAGFDTVEELLRHFNPDTWQEDLETLYTDTAQYRGRN
PDGF-D          TDPTLIADALDKKIAEFDTVEDLLKYFNPESWQEDLENMYLDTPRYRGRS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_015285150.1  FYERKSKVDLDRLNDDVKRYSCTPRNYSVNIREELRVTN.AVFFPRCLLV
PDGF-D          YHDRKSKVDLDRLNDDAKRYSCTPRNYSVNIREELKLAN.VVFFPRCLLV
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXXXPXCVXX

           301                                            350
XP_015285150.1  QRCGGNCGCGFTHWKSCTCTPGKLVKKYHEVLRFIPEAGHIRRRGRTKNN
PDGF-D          QRCGGNCGCGTVNWRSCTCNSGKTVKKYHEVLQFEPG..HIKRRGRAKT.
VEGFC_signature XRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXC

                351                  374
XP_015285150.1  MALIDIQLEHHERCDCVCSSRPPR
PDGF-D          MALVDIQLDHHERCDCICSSRPPR
VEGFC_signature XC~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=GCF46860.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF46860.1/" target="_blank">GCF46860.1</a></a></td><td>GCF46860</td><td>PDGF-C</td><td>hypothetical protein parPi_0006723 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF46860.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF46860.1');">alignment</a><div id="align_GCF46860.1" style="display:none"><pre>


           1                                               50
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      ~~~~~~~~~~~~~~~~~~~~~MCLELLGLGIGMNFGVLFLSLWYLPLIST
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA

           51                                             100
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      KGDPIPEEIYEMLSDSTVRSISDLRRVLRIDSVEENNSSFYLNSTRKHPG
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN

           101                                            150
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXX..XXXCXXXXXXXXXXXXX
GCF46860.1      QNYVPFSRGKRSIVSGGEEPIEPAVIAECKTRAEVFEISRSMVDPTNANF
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRKTQCMPREVCIDVGKEFGVATNT

           151                                            200
VEGFC_signature XXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXXXX.XX
GCF46860.1      IVWPPCVEVQRCTGCCNTRNMHCLPTRVRVRHIQVNKIRFVQKKATFNKA
VEGF-C          FFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQGPK.PV

           201                                            250
VEGFC_signature XXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      IVALEDHLECRCEAVSPRPLSQARSHEHRAAPRLSTVGPLPARARAHRLM
VEGF-C          TISFANHTSCRCMSKLDVYR.QVHSIIRRSLPATLPQCQAANKTCPTNYM

           251                                            300
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      PQKRKHRKFKHVHDKQALKDIFPA~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          WNNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCRAG

           301                                            350
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          LRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRTCP

           351                                            400
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          RNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACEPG

                401               421
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~
GCF46860.1      ~~~~~~~~~~~~~~~~~~~~~
VEGF-C          FSYSEEVCRCVPSYWKRPQMS



</pre></div></td></tr>
<tr><td><a id=GCF52154.1><a href="https://www.ncbi.nlm.nih.gov/protein/GCF52154.1/" target="_blank">GCF52154.1</a></a></td><td>GCF52154</td><td>PDGF-C</td><td>hypothetical protein parPi_0013669 [Paroedura picta]</td><td>None</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=GCF52154.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_GCF52154.1');">alignment</a><div id="align_GCF52154.1" style="display:none"><pre>


           1                                               50
GCF52154.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~MMRRTPAALGLLGFCY
VEGF-C          MHLLGFFSVACSLLAAALLPGPREAPAAAAAFESGLDLSDAEPDAGEATA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
GCF52154.1      LLAALAQEAEIPQEVIDRLAHSEIHSIRDLQRLLEIDSVGYDDASETNLI
VEGF-C          YASKDLEEQLRSVSSVDELMTVLYPEYWKMYKCQLRKGGWQHNREQANLN
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
GCF52154.1      PHGVHVIKHTQDNRPRPIRRKRSIEEAIPAVCKTRTVIYEIPRSQIDPTS
VEGF-C          SRTEETIKFAAAHYNTEILKSIDNEWRK.TQCMPREVCIDVGKEFG..VA
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXX..XX

           151                                            200
GCF52154.1      ANFLIWPPCVEVKRCTGCCNTSSVKCQPSRIHHRSVKVAKVEYVRKKPKL
VEGF-C          TNTFFKPPCVSVYRCGGCCNSEGLQCMNTSTSYLSKTLFEITVPLSQG.P
VEGFC_signature XXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXXXXXXXXXXXXXX.X

           201                                            250
GCF52154.1      KEVLVKLEEHMECTCTSSSSNSDYREEETDVR~~~~~~~~~~~~~~~~~~
VEGF-C          KPVTISFANHTSCRCMSKLDVYRQVHSIIRRSLPATLPQCQAANKTCPTN
VEGFC_signature XXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
GCF52154.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          YMWNNHICRCLAQEDFMFSSDAGDDSTDGFHDICGPNKELDEETCQCVCR
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           301                                            350
GCF52154.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          AGLRPASCGPHKELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCKRT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           351                                            400
GCF52154.1      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          CPRNQPLNPGKCACECTESPQKCLLKGKKFHHQTCSCYRRPCTNRQKACE
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

                401                 423
GCF52154.1      ~~~~~~~~~~~~~~~~~~~~~~~
VEGF-C          PGFSYSEEVCRCVPSYWKRPQMS
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028582005.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028582005.1/" target="_blank">XP_028582005.1</a></a></td><td>XP_028582005</td><td>PDGF-D</td><td>platelet-derived growth factor D [Podarcis muralis]</td><td>PDGF-D</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028582005.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028582005.1');">alignment</a><div id="align_XP_028582005.1" style="display:none"><pre>


           1                                               50
XP_028582005.1  MQPLEHLLFYVAICASFCNRQVSSAGAQSGSIRALRASNLRRDESNHLTA
PDGF-D          ~~MHRLIFVYTLICANFCSCRDTSATPQSASIKALRNANLRRDESNHLTD
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028582005.1  LYRKEETIRVLRSGEIHSPRFPHNYPRNLLLTWRLLSPENARIQLVFDNQ
PDGF-D          LYRRDETIQVKGNGYVQSPRFPNSYPRNLLLTWRLHSQENTRIQLVFDNQ
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           101                                            150
XP_028582005.1  FGLEEPENDICRYDFVEVEDISEISTLIRGRWCGYKEAPPRITSKTNQLK
PDGF-D          FGLEEAENDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIKSRTNQIK
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           151                                            200
XP_028582005.1  ITFKSDDFFVAKPGFKIYYSLVDDLQPAAAETN.WESVTSPVLGVSYHSP
PDGF-D          ITFKSDDYFVAKPGFKIYYSLLEDFQPAAASETNWESVTSSISGVSYNSP
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           201                                            250
XP_028582005.1  SVTDATLTAEALDQTVAGFDTVEELLKHFNPDTWQEDLESLYTDTAPHRG
PDGF-D          SVTDPTLIADALDKKIAEFDTVEDLLKYFNPESWQEDLENMYLDTPRYRG
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           251                                            300
XP_028582005.1  RSFYERKSKVDLDRLNDDVKRYSCTPRNYSVNIREELKQTN.VVFFPRCL
PDGF-D          RSYHDRKSKVDLDRLNDDAKRYSCTPRNYSVNIREELKLAN.VVFFPRCL
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~CXXXXXCXXXXXXXXXXXXXXXXPXCV

           301                                            350
XP_028582005.1  LVQRCGGNCGCGFNNWKSCTCTAGKLVKKYHEVLRFIPEAGHIRRRGRTK
PDGF-D          LVQRCGGNCGCGTVNWRSCTCNSGKTVKKYHEVLQFEPG..HIKRRGRAK
VEGFC_signature XXXRCXGCCXXXXX...XCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

                351                    376
XP_028582005.1  NNMALIDIQLEHHERCDCVCSARPPR
PDGF-D          T.MALVDIQLDHHERCDCICSSRPPR
VEGFC_signature XXXXCXC~~~~~~~~~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_015272715.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_015272715.1/" target="_blank">XP_015272715.1</a></a></td><td>XP_015272715</td><td>PDGF-B</td><td>PREDICTED: platelet-derived growth factor subunit A [Gekko japonicus]</td><td>PDGF-A</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_015272715.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_015272715.1');">alignment</a><div id="align_XP_015272715.1" style="display:none"><pre>


           1                                               50
XP_015272715.1  MRTEAGVWLLGLCYLVAAVAEEAEIPQEVIERLAHSEIHSIRDLQRLLEI
PDGF-A          MRTLACLLLLGCGYLAHVLAEEAEIPREVIERLARSQIHSIRDLQRLLEI
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_015272715.1  DSVGYDDASETNLILHGVHSIKHAQDNRPRPVRRKRSIEEAIPAVCKTRT
PDGF-A          DSVGSEDSLDTSLRAHGVHATKHVPEKRPLPIRRKRSIEEAVPAVCKTRT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXX..

           101                                            150
XP_015272715.1  VIYEIPRSQIDPTSANFLIWPPCVEVKRCTGCCNTSSVKCQPSRIHHRSV
PDGF-A          VIYEIPRSQVDPTSANFLIWPPCVEVKRCTGCCNTSSVKCQPSRVHHRSV
VEGFC_signature XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCX.XXXXXXXX

           151                                            200
XP_015272715.1  KRAFGSKILNGDILLCQSKLIVSVYRQPSGFESAD~~~~~~~~~~~~~~~
PDGF-A          KVAKVEYVRKKPKLKEVQVRLEEHLECACATTSLNPDYREEDTGRPRESG
VEGFC_signature XXXXXXXXXXXXXXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~

                201     211
XP_015272715.1  ~~~~~~~~~~~
PDGF-A          KKRKRKRLKPT
VEGFC_signature ~~~~~~~~~~~



</pre></div></td></tr>
<tr><td><a id=XP_028561875.1><a href="https://www.ncbi.nlm.nih.gov/protein/XP_028561875.1/" target="_blank">XP_028561875.1</a></a></td><td>XP_028561875</td><td>PDGF-B</td><td>platelet-derived growth factor subunit A [Podarcis muralis]</td><td>PDGF-A</td><td><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?LAYOUT=OneWindow&PROGRAM=blastp&PAGE=Proteins&CMD=Web&DATABASE=nr&FORMAT_TYPE=HTML&NCBI_GI=on&SHOW_OVERVIEW=yes&QUERY=XP_028561875.1" target="_blank">blastp</a></td><td><a href="#" onclick="toggle_visibility('align_XP_028561875.1');">alignment</a><div id="align_XP_028561875.1" style="display:none"><pre>


           1                                               50
XP_028561875.1  MRTWAGFWLIGLCYLALVLAEEAEIPQEVIERLAHSEIHSIRDLQRLLEI
PDGF-A          MRTLACLLLLGCGYLAHVLAEEAEIPREVIERLARSQIHSIRDLQRLLEI
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

           51                                             100
XP_028561875.1  DSVGYDDPSETNLIPHSIHSVKHGQENRPRPIRRKRSVEEAIPAVCKTRT
PDGF-A          DSVGSEDSLDTSLRAHGVHATKHVPEKRPLPIRRKRSIEEAVPAVCKTRT
VEGFC_signature ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~CXX..

           101                                            150
XP_028561875.1  VIYEIPRSQIDPTSANFLIWPPCVEVKRCTGCCNTSSVKCQPSRIHHRNV
PDGF-A          VIYEIPRSQVDPTSANFLIWPPCVEVKRCTGCCNTSSVKCQPSRVHHRSV
VEGFC_signature XXXCXXXXXXXXXXXXXXXXPXCVXXXRCXGCCXXXXXXCXXXXXXXXXX

           151                                            200
XP_028561875.1  KVRRNGTQVQRCGRVGVLREEGVQHFVRILSFSFFSPWQPREVIRLFLF~
PDGF-A          KVAKVEYVRKKPKLKEVQVRLEEHLECACATTSLNPDYREEDTGRPRESG
VEGFC_signature XXXXXXXXXXX.XXXXXXXXXXXXXXCXC~~~~~~~~~~~~~~~~~~~~~

                201     211
XP_028561875.1  ~~~~~~~~~~~
PDGF-A          KKRKRKRLKPT
VEGFC_signature ~~~~~~~~~~~



</pre></div></td></tr>
</table>
<pre>
<p><object type="image/svg+xml" data="../alignments_and_trees/lepidosauria_excl._toxicofera.svg">Here should be the svg tree!</object></p><html>
<style>
SPAN { font-family: courier new, courier-new, courier, monospace; font-weight: bold; font-size: 11pt;}
SPAN { line-height:100%}
SPAN {	white-space: pre}
SPAN.value0 {background: #9B92FF}
SPAN.value1 {background: #B4FFB4}
SPAN.value2 {background: #BEFFBE}
SPAN.value3 {background: #C8FFC8}
SPAN.value4 {background: #FFFFB7}
SPAN.value5 {background: #FFFFAD}
SPAN.value6 {background: #FFFFAD}
SPAN.value7 {background: #FFE6E6}
SPAN.value8 {background: #FFDCDC}
SPAN.value9 {background: #FFC8C8}
SPAN.valuedefault {}
SPAN.valuegap {}
SPAN.valueink {background: 000000}
</style><body><span class=valuedefault>T-COFFEE,&nbsp;Version_11.00.8cbe486&nbsp;(2014-08-12&nbsp;22:05:29&nbsp;-&nbsp;Revision&nbsp;8cbe486&nbsp;-&nbsp;Build&nbsp;477)</span><br><span class=valuedefault>Cedric&nbsp;Notredame&nbsp;</span><br><span class=valuedefault>CPU&nbsp;TIME:0&nbsp;sec.</span><br><span class=valuedefault>SCORE=874</span><br><span class=valuedefault>*</span><br><span class=valuedefault>&nbsp;</span><span class=value0>B</span><span class=value1>A</span><span class=value2>D</span><span class=value3>&nbsp;</span><span class=value4>A</span><span class=value5>V</span><span class=value6>G</span><span class=value7>&nbsp;</span><span class=value8>G</span><span class=value9>OOD</span><br><span class=valuedefault>*</span><br><span class=value8>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;83</span><br><span class=value8>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;85</span><br><span class=value8>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;85</span><br><span class=value8>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;82</span><br><span class=value8>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;81</span><br><span class=value8>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;82</span><br><span class=value8>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;80</span><br><span class=value9>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;95</span><br><span class=value9>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;99</span><br><span class=value8>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;83</span><br><span class=value8>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;83</span><br><span class=value8>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value9>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;90</span><br><span class=value9>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value8>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;81</span><br><span class=value9>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;91</span><br><span class=value9>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value8>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;85</span><br><span class=value9>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;94</span><br><span class=value9>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;94</span><br><span class=value9>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;94</span><br><span class=value9>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;94</span><br><span class=value9>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;94</span><br><span class=value9>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value8>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;89</span><br><span class=value9>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;90</span><br><span class=value8>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value8>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;88</span><br><span class=value8>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value7>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;76</span><br><span class=value7>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;79</span><br><span class=value7>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;78</span><br><span class=value7>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;78</span><br><span class=value8>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;80</span><br><span class=value8>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value8>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;80</span><br><span class=value8>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value8>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value7>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;79</span><br><span class=value7>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;78</span><br><span class=value8>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value7>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;75</span><br><span class=value8>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;84</span><br><span class=value8>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;81</span><br><span class=value9>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;92</span><br><span class=value9>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value9>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;93</span><br><span class=value8>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;87</span><br><span class=value8>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;89</span><br><span class=value8>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;89</span><br><span class=value8>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;88</span><br><span class=value8>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;80</span><br><span class=value7>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;79</span><br><span class=value8>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=value8>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;86</span><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuedefault>:&nbsp;&nbsp;87</span><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>MLLS</span><span class=value6>G</span><span class=valuegap>----</span><span class=value8>FLWLTFALVS</span><span class=value7>Q</span><span class=value8>RQ</span><span class=valuegap>-</span><span class=value8>GA</span><span class=value7>RA</span><span class=value6>E</span><span class=value7>SNLSSKFQ</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value8>MLLF</span><span class=value7>G</span><span class=valuegap>----</span><span class=value8>FLL</span><span class=value7>LTFALVS</span><span class=value6>QR</span><span class=value5>Q</span><span class=valuegap>-</span><span class=value7>GAEA</span><span class=value6>E</span><span class=value8>SNLSSKFQ</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value8>MLLF</span><span class=value7>G</span><span class=valuegap>----</span><span class=value8>FFL</span><span class=value7>LTSALVS</span><span class=value6>PR</span><span class=value5>Q</span><span class=valuegap>-</span><span class=value7>GAQG</span><span class=value6>E</span><span class=value8>SNLSSKFQ</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value9>MQLP</span><span class=value7>A</span><span class=value6>PFLL</span><span class=value9>YVV</span><span class=value7>AWANFCH</span><span class=value6>RH</span><span class=valuegap>--</span><span class=value7>GSAA</span><span class=value8>QSGSIRALR</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value9>MQLP</span><span class=value7>A</span><span class=value6>PFLL</span><span class=value9>YVV</span><span class=value7>AWANFCH</span><span class=value6>RH</span><span class=valuegap>--</span><span class=value7>GSAA</span><span class=value8>QSGSIRALR</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value9>MQLP</span><span class=value7>A</span><span class=value6>PFLL</span><span class=value9>YVV</span><span class=value7>AWANFCH</span><span class=value6>RH</span><span class=valuegap>--</span><span class=value7>GSAA</span><span class=value8>QSGSIRALR</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value9>MQPL</span><span class=value7>E</span><span class=value6>HLLF</span><span class=value9>YVA</span><span class=value7>ICASFCN</span><span class=value6>RQ</span><span class=value9>VS</span><span class=value7>SAGA</span><span class=value8>QSGSIRALR</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>MSLF</span><span class=value7>G</span><span class=valuegap>----</span><span class=value8>LLL</span><span class=value7>LTSALAG</span><span class=value6>QR</span><span class=value5>Q</span><span class=valuegap>-</span><span class=value7>GT</span><span class=value6>QAE</span><span class=value8>SNLSSKFQ</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>MH</span><span class=value7>R</span><span class=value6>LIFV</span><span class=value9>YTL</span><span class=value7>ICANFCS</span><span class=value6>CR</span><span class=value9>DT</span><span class=value7>SAT</span><span class=value6>P</span><span class=value8>QSASIKALR</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;</span><span class=value7>&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>FSGGKEQNG</span><span class=valuegap>---</span><span class=value9>V</span><span class=value8>QEP</span><span class=value7>HHEKIITVSANGSIHSPKFP</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value8>FSSAKEQNG</span><span class=valuegap>---</span><span class=value8>VQEP</span><span class=value7>Q</span><span class=value6>HEKIITVSANGSIHSPKF</span><span class=value5>P</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value8>FSSAKEQNG</span><span class=valuegap>---</span><span class=value8>VQEP</span><span class=value7>Q</span><span class=value6>HEKIITVSANGSIHSPKFP</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value8>ASNLRRDES</span><span class=value9>NHLTALY</span><span class=value7>R</span><span class=value6>KEETINVRGNGHIHSP</span><span class=value5>RFP</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value8>ASNLRRDES</span><span class=value9>NHLTALY</span><span class=value7>R</span><span class=value6>KEETINVRGNGHIHSP</span><span class=value5>RFP</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value8>ASNLRRD</span><span class=valuegap>------</span><span class=value9>ALY</span><span class=value7>R</span><span class=value6>KEETINVRGNGHIHSP</span><span class=value5>RFP</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value8>ASNLRRDES</span><span class=value9>NHLTALY</span><span class=value7>R</span><span class=value6>KEETIRVLRSGEIHSP</span><span class=value5>RFP</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>FSSNKEQNG</span><span class=valuegap>---</span><span class=value8>V</span><span class=value7>QDPQ</span><span class=value6>HERIITVSTNGSIHSPR</span><span class=value5>FP</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>NANLRRDES</span><span class=value9>NHLTDLY</span><span class=value6>RRDETIQVKGNGYVQSP</span><span class=value5>RFP</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value8>&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>FTYPRNTVLVW</span><span class=value9>RLVAIQENVLIQLTFDERFGLEDPE</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value5>YTYP</span><span class=value4>RNTVLVW</span><span class=value9>RLVAIEENVLIQLTFDERFGLEDPE</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value5>YTYP</span><span class=value4>RNTVLVW</span><span class=value9>RLVAVEENVLIQLTFDERFGLEDPE</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value5>NSYP</span><span class=value4>RNL</span><span class=value3>LLTW</span><span class=value9>KLLS</span><span class=valuegap>-</span><span class=value9>PENTRIQLVFDSQFGLEEPE</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value5>NSYP</span><span class=value4>RNL</span><span class=value3>LLTW</span><span class=value9>KLLS</span><span class=valuegap>-</span><span class=value9>PENTRIQLVFDSQFGLEEPE</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value5>NSYP</span><span class=value4>RNL</span><span class=value3>LLTW</span><span class=value9>KLLS</span><span class=valuegap>-</span><span class=value9>PENTRIQLVFDSQFGLEEPE</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value5>HNYP</span><span class=value4>RNL</span><span class=value3>LLTW</span><span class=value9>RLLS</span><span class=valuegap>-</span><span class=value9>PENARIQLVFDNQFGLEEPE</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>HTYP</span><span class=value4>RNTVLVW</span><span class=value9>RLVAVEENVWIQLTFDERFGLEDPE</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>NSYP</span><span class=value4>RNL</span><span class=value3>LLTW</span><span class=value9>RLHS</span><span class=valuegap>-</span><span class=value9>QENTRIQLVFDNQFGLEEAE</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value5>&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>DDICKYDFVEIEEPSDGSIL</span><span class=valuegap>--</span><span class=value9>GRWCGSTAVPGKQI</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value9>DDICKYDFVEVEEPSDGSIL</span><span class=valuegap>--</span><span class=value9>GRWCGSTAVPGKQI</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value9>DDICKYDFVEVEEPSDGTIL</span><span class=valuegap>--</span><span class=value9>GRWCGSTTVPGKQI</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value9>NEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRIT</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------</span><span class=value5>MGIV</span><span class=value9>VH</span><span class=value5>GHLEDHR</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value9>NEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRIT</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value9>NEICRYDFVEVEDISEISTIIRGRWCGHKEVPPRIT</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value9>NDICRYDFVEVEDISEISTLIRGRWCGYKEAPPRIT</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>DDICKYDFVEVEEPSDGTIL</span><span class=valuegap>--</span><span class=value9>GRWCGSGTVPGKQI</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>NDICRYDFVEVEDISETSTIIRGRWCGHKEVPPRIK</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>SKGNQIRIRFVSDEYFPSEPGFCIHYTLLTPHQT</span><span class=value8>ES</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SKGNQIRIRFVSDEYFPSEPGFCIHYTLLTPHQTE</span><span class=value8>S</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SKGNQIRIRFVSDEYFPSEPGFCIHYTLLVP</span><span class=valuegap>--</span><span class=value9>TE</span><span class=value8>S</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------------</span><span class=value5>MRAAN</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------------------------</span><span class=value8>MCLEL</span><span class=value7>L</span><span class=value6>G</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------------------------</span><span class=value8>MCLEL</span><span class=value7>L</span><span class=value6>G</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------------------------</span><span class=value8>MCLEL</span><span class=value7>L</span><span class=value6>G</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------------------------</span><span class=value8>MCLEL</span><span class=value7>L</span><span class=value6>G</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SRTNQLKITFKSDDYFVAKPGFKIYYSRV</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------</span><span class=value5>LTEGDDLQP</span><span class=value4>A</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SRTNQLKITFKSDDYFVAKPGFKIYYSRVDDLQP</span><span class=value8>A</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SRTNQLKITFKSDDYFVAKPGFKIYYSRVDDLQP</span><span class=value8>A</span><span class=valuegap>-</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------------------------</span><span class=value8>MCLEL</span><span class=value7>LG</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value9>SKTNQLKITFKSDDFFVAKPGFKIYYSLVDDLQP</span><span class=value8>A</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>SKGNQIRIRFVSDEYFPSEPGFCIHYNIVMPQFTE</span><span class=value8>A</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>SRTNQIKITFKSDDYFVAKPGFKIYYSLLEDFQPA</span><span class=value7>A</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value8>MHLL</span><span class=value6>G</span><span class=value8>FFSLGCYLAAGAVVLLGA</span><span class=value7>R</span><span class=value6>EP</span><span class=value2>A</span><span class=value4>TA</span><span class=value5>AA</span><span class=value4>YESG</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value8>MHLL</span><span class=value6>G</span><span class=value8>FFSLGCYLAAGAVVLLGA</span><span class=value7>R</span><span class=value6>EP</span><span class=value2>A</span><span class=value4>TA</span><span class=value5>AA</span><span class=value4>YESG</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value8>MHLL</span><span class=value7>G</span><span class=value8>VLSLGCSLAAATLGLLGA</span><span class=value7>R</span><span class=value6>EP</span><span class=value2>A</span><span class=value4>TAAAYESG</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------</span><span class=value3>MVSEDG</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------</span><span class=value6>MVVGC</span><span class=valuegap>----------</span><span class=value7>KC</span><span class=value6>RPP</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------</span><span class=value6>MVVGC</span><span class=valuegap>----------</span><span class=value6>KCRP</span><span class=value4>P</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value8>MY</span><span class=value7>KL</span><span class=valuegap>-</span><span class=value7>WTT</span><span class=value8>VNV</span><span class=value7>FMV</span><span class=value8>SFLHLLQGS</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value7>MYK</span><span class=value6>L</span><span class=valuegap>-</span><span class=value6>WTT</span><span class=value7>VN</span><span class=value6>VFMVSLLH</span><span class=value7>L</span><span class=value6>LQ</span><span class=value7>G</span><span class=value6>Y</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------</span><span class=value7>MLGPIDGE</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value7>MY</span><span class=value6>K</span><span class=value5>P</span><span class=valuegap>-</span><span class=value5>WTTVN</span><span class=value6>I</span><span class=value5>FIVSFLH</span><span class=value6>LLQGS</span><span class=valuegap>------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------</span><span class=value7>SWT</span><span class=value3>G</span><span class=value8>SR</span><span class=value7>EPAADQ</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value8>A</span><span class=value9>SPTV</span><span class=value8>LPP</span><span class=valuegap>---------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value8>A</span><span class=value9>SPTV</span><span class=value8>LPP</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value8>A</span><span class=value9>SPTI</span><span class=value8>LPP</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value4>GGT</span><span class=valuegap>--</span><span class=value4>AKQASFPQRLLR</span><span class=value3>EEVKD</span><span class=value7>R</span><span class=value6>RMG</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>IG</span><span class=value7>V</span><span class=valuegap>--</span><span class=value7>GMNFGVLFVCFSYLPLVS</span><span class=value6>SEG</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>IG</span><span class=value7>V</span><span class=valuegap>--</span><span class=value7>GMNFGVLFVCFSYLPLVS</span><span class=value6>SEG</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>IG</span><span class=value7>V</span><span class=valuegap>--</span><span class=value7>GMNFGVLFVCFSYLPLVS</span><span class=value6>SEG</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>IG</span><span class=value7>I</span><span class=valuegap>--</span><span class=value7>GMNFGVLFLSLWYLPLIS</span><span class=value6>TKG</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------</span><span class=value9>GVSFHSPSV</span><span class=value8>TDP</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>A</span><span class=value3>SETN</span><span class=valuegap>-</span><span class=value3>W</span><span class=valuegap>-</span><span class=value4>ESVTS</span><span class=value5>SVSGVSFHSP</span><span class=value4>SVTD</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value5>A</span><span class=value9>SETN</span><span class=valuegap>-</span><span class=value9>WESVTSPVLGVSFHSPSV</span><span class=value8>TDP</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value5>A</span><span class=value9>SETN</span><span class=valuegap>-</span><span class=value9>WESVTSPVLGVSFHSPSV</span><span class=value8>TDP</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>LG</span><span class=value7>I</span><span class=valuegap>--</span><span class=value7>GMNFGVLFLSLWYLPLIS</span><span class=value6>TKG</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>M</span><span class=value5>M</span><span class=value6>R</span><span class=valuegap>--</span><span class=value7>RTPAALGLLGFCYLLAAL</span><span class=value6>AQE</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value5>A</span><span class=value9>AETN</span><span class=valuegap>-</span><span class=value9>WESVTSPVLGVSYHSPSV</span><span class=value8>TDA</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value7>M</span><span class=valuegap>--</span><span class=value7>RTEAGVWLLGLCYLVAAV</span><span class=value6>AEE</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value7>M</span><span class=valuegap>--</span><span class=value7>RTWAGFWLIGLCYLALVL</span><span class=value5>AEE</span><span class=valuegap>---------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value8>MHLL</span><span class=value6>G</span><span class=value8>FFSVACSLLAAALLPGPR</span><span class=value7>EA</span><span class=value6>P</span><span class=value2>A</span><span class=value4>AA</span><span class=value5>AAF</span><span class=value4>ESG</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>MYR</span><span class=value5>E</span><span class=valuegap>-</span><span class=value6>WVVVNVFMMLYVQLVQGS</span><span class=value1>SN</span><span class=valuegap>----------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value7>M</span><span class=valuegap>--</span><span class=value7>RTLACLLLLGCGYLAHVL</span><span class=value6>AEE</span><span class=valuegap>---------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value7>M</span><span class=valuegap>--</span><span class=value7>NRCWALFLSLCCYLRLVS</span><span class=value6>AEG</span><span class=valuegap>---------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value8>V</span><span class=value9>SPSV</span><span class=value7>LPP</span><span class=valuegap>---------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>A</span><span class=value9>SETN</span><span class=valuegap>-</span><span class=value9>WESVTSSISGVSYNSPSV</span><span class=value8>TDP</span><span class=valuegap>---------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value3>&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value6>&nbsp;</span><span class=value5>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;</span><span class=value1>&nbsp;</span><span class=value2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value4>QGYYEEE</span><span class=valuegap>-------------</span><span class=value4>PDVG</span><span class=valuegap>-</span><span class=value3>DAK</span><span class=value1>T</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value4>QGYYEEE</span><span class=valuegap>-------------</span><span class=value4>PDVG</span><span class=valuegap>-</span><span class=value3>DAK</span><span class=value2>Q</span><span class=valuegap>------</span><span class=value4>T</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>QGYYEE</span><span class=value3>E</span><span class=valuegap>-------------</span><span class=value4>PEL</span><span class=value3>L</span><span class=value2>AEAK</span><span class=value1>Q</span><span class=valuegap>------</span><span class=value4>A</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value3>I</span><span class=value2>DHFLR</span><span class=valuegap>-----------------------------</span><span class=value4>A</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------</span><span class=value2>RDQQ</span><span class=valuegap>------</span><span class=value4>T</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------</span><span class=value2>RDQ</span><span class=valuegap>-------</span><span class=value4>T</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value8>ES</span><span class=valuegap>-------------</span><span class=value8>DSV</span><span class=valuegap>---</span><span class=value4>KKP</span><span class=valuegap>-----</span><span class=value8>S</span><span class=value5>L</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value4>GSV</span><span class=valuegap>---</span><span class=value2>KRA</span><span class=valuegap>-----</span><span class=value4>SL</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value5>T</span><span class=value4>T</span><span class=valuegap>-------------</span><span class=value5>EEV</span><span class=value2>L</span><span class=valuegap>-</span><span class=value2>EERT</span><span class=valuegap>-----</span><span class=value4>SL</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value3>EH</span><span class=valuegap>-------------</span><span class=value4>GSV</span><span class=valuegap>---</span><span class=value2>KRT</span><span class=valuegap>-----</span><span class=value4>SL</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value2>MPSSAD</span><span class=valuegap>-</span><span class=value2>SGA</span><span class=valuegap>-----</span><span class=value2>G</span><span class=value3>R</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value2>MPSSAD</span><span class=valuegap>-</span><span class=value2>SGA</span><span class=valuegap>-----</span><span class=value2>G</span><span class=value3>R</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value2>MPSSAD</span><span class=valuegap>-</span><span class=value2>SGA</span><span class=valuegap>-----</span><span class=value2>G</span><span class=value3>R</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value8>MC</span><span class=value9>C</span><span class=value8>R</span><span class=value7>NRG</span><span class=value4>G</span><span class=value2>IASRSE</span><span class=valuegap>-</span><span class=value2>SEL</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>SEPEPE</span><span class=value8>PA</span><span class=value2>S</span><span class=value3>R</span><span class=value2>S</span><span class=value8>AA</span><span class=value9>C</span><span class=value8>GRFGP</span><span class=value2>EPASGA</span><span class=valuegap>-</span><span class=value2>P</span><span class=value1>EAG</span><span class=value2>L</span><span class=value1>APG</span><span class=value7>GC</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>S</span><span class=value9>A</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>S</span><span class=value9>A</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>S</span><span class=value9>A</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value4>P</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value4>P</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value4>P</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value4>P</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value5>P</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>T</span><span class=valuegap>-</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value4>P</span><span class=value3>T</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>T</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>T</span><span class=valuegap>-</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value4>P</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>A</span><span class=value5>E</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>T</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>A</span><span class=value5>E</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value7>A</span><span class=value6>E</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>LDLSDAE</span><span class=valuegap>-------------</span><span class=value4>PDAG</span><span class=valuegap>-</span><span class=value4>E</span><span class=value3>AT</span><span class=value1>A</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value2>EH</span><span class=valuegap>-------------</span><span class=value2>GPV</span><span class=valuegap>---</span><span class=value1>KR</span><span class=value2>S</span><span class=valuegap>-----</span><span class=value8>SQ</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>A</span><span class=value5>E</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value6>D</span><span class=value5>P</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value9>SA</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value8>T</span><span class=valuegap>-</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value8>&nbsp;</span><span class=value7>&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;</span><span class=value3>&nbsp;</span><span class=value1>&nbsp;</span><span class=value2>&nbsp;&nbsp;&nbsp;</span><span class=value1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value4>&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value5>HA</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value5>HA</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>YA</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value5>Y</span><span class=value9>A</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value5>HA</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value5>HA</span><span class=value6>SKE</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>SA</span><span class=value5>SEW</span><span class=value6>SEQ</span><span class=valuegap>-</span><span class=value7>QIRKAA</span><span class=value6>SLEEL</span><span class=value5>LR</span><span class=value2>I</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value5>SA</span><span class=value6>LEW</span><span class=value8>SEQ</span><span class=valuegap>-</span><span class=value8>QIRKAA</span><span class=value6>SLEE</span><span class=value5>LLR</span><span class=value2>I</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value5>SA</span><span class=value6>LEW</span><span class=value8>SEQ</span><span class=valuegap>-</span><span class=value8>QIRKAA</span><span class=value6>SLEE</span><span class=value5>LLR</span><span class=value2>I</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value5>SA</span><span class=value6>LEW</span><span class=value8>SEQ</span><span class=valuegap>-</span><span class=value8>QIRKAA</span><span class=value6>SLEE</span><span class=value5>LLR</span><span class=value2>I</span><span class=valuegap>-------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value3>AW</span><span class=valuegap>---------------</span><span class=value4>TS</span><span class=value3>LLQ</span><span class=value2>H</span><span class=value1>RSSSRSR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value3>AW</span><span class=valuegap>---------------</span><span class=value4>TS</span><span class=value3>LLQ</span><span class=value2>H</span><span class=value1>RSSSRSR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value3>AW</span><span class=valuegap>---------------</span><span class=value4>TS</span><span class=value3>LLQ</span><span class=value2>H</span><span class=value1>RSSSRSR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------</span><span class=value2>ETGAREKR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LW</span><span class=valuegap>---------------</span><span class=value8>EP</span><span class=value2>RVPRQGAPET</span><span class=value3>RRP</span><span class=value1>GRW</span><span class=value2>C</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------</span><span class=value2>MTKLGNGN</span><span class=value1>EDL</span><span class=valuegap>------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>FS</span><span class=value6>LDL</span><span class=value8>LNN</span><span class=valuegap>-</span><span class=value8>AV</span><span class=value7>AGFS</span><span class=value6>TVEE</span><span class=value5>LIR</span><span class=valuegap>--------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value9>FS</span><span class=value6>LDL</span><span class=value8>LNN</span><span class=valuegap>-</span><span class=value8>AV</span><span class=value7>AGFS</span><span class=value6>TVEE</span><span class=value5>LIR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value9>LP</span><span class=value6>LEL</span><span class=value8>LNN</span><span class=valuegap>-</span><span class=value8>AVA</span><span class=value7>GFS</span><span class=value6>TVEE</span><span class=value5>LIK</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYELL</span><span class=value7>SDSSLRSIR</span><span class=value5>DLQQ</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYELL</span><span class=value7>SDSSLRSIR</span><span class=value5>DLQQ</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYELL</span><span class=value7>SDSSLRSIR</span><span class=value5>DLQQ</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYELL</span><span class=value7>SDSSLRSIR</span><span class=value5>DLQQ</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYEML</span><span class=value7>SDSSVRSIS</span><span class=value5>DLRR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value9>LT</span><span class=value6>AEA</span><span class=value8>LDQ</span><span class=valuegap>-</span><span class=value8>TIA</span><span class=value7>GFD</span><span class=value6>TVEE</span><span class=value5>LLR</span><span class=valuegap>--------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>LT</span><span class=value6>AEA</span><span class=value8>LDQ</span><span class=valuegap>-</span><span class=value8>TIA</span><span class=value7>GFD</span><span class=value6>TVEE</span><span class=value5>LLR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value9>LT</span><span class=value6>AEA</span><span class=value8>LDQ</span><span class=valuegap>-</span><span class=value8>TIA</span><span class=value7>GFD</span><span class=value6>TVEE</span><span class=value5>LLR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value9>LT</span><span class=value6>AEA</span><span class=value8>LDQ</span><span class=valuegap>-</span><span class=value8>TIA</span><span class=value7>GFD</span><span class=value6>TVEE</span><span class=value5>LLR</span><span class=valuegap>--------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEEIYEM</span><span class=value6>L</span><span class=value7>SDSTVRSIS</span><span class=value5>DLRR</span><span class=valuegap>--------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>IPQEVIDRL</span><span class=value7>AHSEIHSIR</span><span class=value5>DLQR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value9>LT</span><span class=value6>AEA</span><span class=value8>LDQ</span><span class=valuegap>-</span><span class=value8>TVA</span><span class=value7>GFD</span><span class=value6>T</span><span class=value5>V</span><span class=value6>EE</span><span class=value5>LLK</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=value5>IPQEVIERL</span><span class=value7>AHSEIHSIR</span><span class=value5>DLQR</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=value6>IPQ</span><span class=value5>EVI</span><span class=value6>ERL</span><span class=value8>AHSEIHSIR</span><span class=value6>D</span><span class=value5>L</span><span class=value6>QR</span><span class=valuegap>--------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>YA</span><span class=value6>SKD</span><span class=value8>LEE</span><span class=valuegap>-</span><span class=value8>QLRSVS</span><span class=value6>SVDE</span><span class=value5>LMT</span><span class=value2>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>ST</span><span class=value6>LER</span><span class=value8>SEQ</span><span class=valuegap>-</span><span class=value8>QIRAAS</span><span class=value6>SLEE</span><span class=value5>LLR</span><span class=value2>I</span><span class=valuegap>-------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>IPREVIERL</span><span class=value7>ARSQIHSIR</span><span class=value5>DLQR</span><span class=valuegap>--------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>IPEELYEML</span><span class=value7>SDHSIRSFD</span><span class=value5>DLQR</span><span class=valuegap>--------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>LP</span><span class=value6>LDL</span><span class=value8>LNN</span><span class=valuegap>-</span><span class=value8>AI</span><span class=value7>TAFS</span><span class=value6>TLED</span><span class=value5>LIR</span><span class=valuegap>--------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>LI</span><span class=value6>ADA</span><span class=value8>LDK</span><span class=valuegap>-</span><span class=value8>KIA</span><span class=value7>EFD</span><span class=value6>T</span><span class=value5>V</span><span class=value6>ED</span><span class=value5>LLK</span><span class=valuegap>--------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value4>&nbsp;</span><span class=value5>&nbsp;&nbsp;</span><span class=value1>&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>LYPEYW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GSW</span><span class=value5>QH</span><span class=value7>NRE</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>LYPEYW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GSW</span><span class=value5>QH</span><span class=value7>NRE</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>LYPEYW</span><span class=value5>KM</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RKGS</span><span class=value7>W</span><span class=value5>QH</span><span class=value6>S</span><span class=value7>RE</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>LYPE</span><span class=value7>YW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GSW</span><span class=value5>QH</span><span class=value7>SRE</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>LYPEYW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GSWQH</span><span class=value7>NRE</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>LYPEYW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>FK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GSWQH</span><span class=value7>NRE</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>THSEDW</span><span class=value5>K</span><span class=value6>L</span><span class=valuegap>--</span><span class=value9>W</span><span class=value8>KCRLKL</span><span class=value4>KS</span><span class=value7>V</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>THSEDW</span><span class=value5>KL</span><span class=valuegap>--</span><span class=value8>WKCRLKL</span><span class=value4>KSV</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>THSEDW</span><span class=value5>KL</span><span class=valuegap>--</span><span class=value8>WKCRLKL</span><span class=value4>KGM</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>THSEDW</span><span class=value5>KL</span><span class=valuegap>--</span><span class=value8>WKCRLKL</span><span class=value4>KGM</span><span class=valuegap>-----</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>T</span><span class=value8>KR</span><span class=value9>MPVPG</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>T</span><span class=value8>KR</span><span class=value9>MPVPG</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>T</span><span class=value8>KR</span><span class=value9>MPVPG</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>R</span><span class=value8>EPAPAM</span><span class=value7>Q</span><span class=value5>L</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWI</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAAWL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWI</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWL</span><span class=value6>H</span><span class=valuegap>--</span><span class=value8>WGLAA</span><span class=value7>LL</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWI</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------------</span><span class=value8>MP</span><span class=value6>T</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAA</span><span class=value7>LLY</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWM</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWM</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>GCRWAG</span><span class=value8>H</span><span class=value1>ARG</span><span class=value5>L</span><span class=value6>RSVLGWK</span><span class=value1>SE</span><span class=value2>R</span><span class=value7>AAE</span><span class=value8>PAMP</span><span class=value6>T</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAA</span><span class=value7>LLY</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWM</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>MNFLLTW</span><span class=value8>L</span><span class=value6>H</span><span class=valuegap>--</span><span class=value7>WGLAA</span><span class=value6>LLY</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWM</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value2>E</span><span class=value6>LSLGRPG</span><span class=value4>G</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWM</span><span class=value6>H</span><span class=valuegap>--</span><span class=value8>WGLAA</span><span class=value7>LL</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLAWL</span><span class=value7>R</span><span class=valuegap>--</span><span class=value8>WGLAA</span><span class=value7>LLY</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAAL</span><span class=value7>LY</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFXLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLXALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLTWL</span><span class=value7>H</span><span class=valuegap>--</span><span class=value8>WGLAALL</span><span class=value7>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value6>M</span><span class=value7>NAP</span><span class=value6>CNAL</span><span class=value5>Q</span><span class=valuegap>--</span><span class=value7>LL</span><span class=value8>LAT</span><span class=value7>TL</span><span class=value6>Q</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value6>M</span><span class=value7>PAP</span><span class=value6>CGAL</span><span class=value5>Q</span><span class=valuegap>--</span><span class=value7>LL</span><span class=value8>LAT</span><span class=value7>TL</span><span class=value6>Q</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>Y</span><span class=value6>LEPD</span><span class=value7>RW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>LDLEDLY</span><span class=valuegap>--------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>Y</span><span class=value6>LEPD</span><span class=value7>RW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>LDLEDLY</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value6>M</span><span class=value7>NAP</span><span class=value6>CGA</span><span class=value5>L</span><span class=value4>Q</span><span class=valuegap>--</span><span class=value6>LL</span><span class=value8>L</span><span class=value7>ATTL</span><span class=value6>Q</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>Y</span><span class=value7>LEPDRW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>LDLEDLY</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LR</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>EDNST</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LR</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>EDNST</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LR</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>EDNST</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LR</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>EDNST</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LQ</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>EDGSN</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>H</span><span class=value6>FNPD</span><span class=value7>TW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLETLY</span><span class=valuegap>--------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>H</span><span class=value6>FNPD</span><span class=value7>TW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLETLY</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>H</span><span class=value6>FNPD</span><span class=value7>TW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLETLY</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>H</span><span class=value6>FNPD</span><span class=value7>TW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLETLY</span><span class=valuegap>--------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>V</span><span class=value5>LR</span><span class=value6>IDSVE</span><span class=valuegap>---</span><span class=value6>ENNSS</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>L</span><span class=value5>LE</span><span class=value6>IDSVG</span><span class=valuegap>---</span><span class=value6>YDD</span><span class=value7>AS</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>H</span><span class=value7>FNPDTW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLESLY</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>L</span><span class=value5>LE</span><span class=value6>IDSVG</span><span class=valuegap>---</span><span class=value6>YDD</span><span class=value7>AS</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value4>LLE</span><span class=value5>IDSVG</span><span class=valuegap>---</span><span class=value7>YDD</span><span class=value6>PS</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>LYPEYW</span><span class=value6>K</span><span class=value5>M</span><span class=valuegap>--</span><span class=value8>YK</span><span class=value9>CQL</span><span class=value8>RK</span><span class=value6>GGWQH</span><span class=value7>NRE</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>THSEDW</span><span class=value5>KL</span><span class=valuegap>--</span><span class=value8>WRCRLRL</span><span class=value4>KSF</span><span class=valuegap>-----</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLSWV</span><span class=value6>H</span><span class=valuegap>--</span><span class=value8>WSLAL</span><span class=value7>LL</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLSWV</span><span class=value6>H</span><span class=valuegap>--</span><span class=value8>WSLAL</span><span class=value7>LL</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value7>M</span><span class=value8>NFLLSWV</span><span class=value6>H</span><span class=valuegap>--</span><span class=value8>WSLAL</span><span class=value7>LL</span><span class=value6>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value6>M</span><span class=value7>SPLLR</span><span class=valuegap>--</span><span class=value4>R</span><span class=valuegap>--</span><span class=value8>LLLAA</span><span class=value7>LL</span><span class=value6>Q</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value6>M</span><span class=value7>SPLLR</span><span class=valuegap>--</span><span class=value4>R</span><span class=valuegap>--</span><span class=value8>LLLAA</span><span class=value7>LL</span><span class=value6>Q</span><span class=valuegap>-------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value2>M</span><span class=value7>PVMRLF</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------</span><span class=value6>MAA</span><span class=value5>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>L</span><span class=value5>LE</span><span class=value6>IDSVG</span><span class=valuegap>---</span><span class=value6>SED</span><span class=value7>SL</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value3>L</span><span class=value5>LH</span><span class=value6>GDPGE</span><span class=valuegap>---</span><span class=value6>EDGAE</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value4>Y</span><span class=value6>LEPERW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>LDLEDLY</span><span class=valuegap>--------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value5>Y</span><span class=value6>FNPESW</span><span class=valuegap>---</span><span class=value3>Q</span><span class=value8>EDLENMY</span><span class=valuegap>--------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value0>&nbsp;</span><span class=value2>&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value5>&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value3>S</span><span class=value6>SFDAR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value5>E</span><span class=value6>DSNP</span><span class=value9>VK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value3>S</span><span class=value6>SFDAR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value5>E</span><span class=value6>DSNP</span><span class=value9>VK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value2>S</span><span class=value6>SFDAR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value4>E</span><span class=value6>DSNP</span><span class=value9>IK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value3>S</span><span class=value6>SFIAR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value4>E</span><span class=value6>DSNP</span><span class=value9>IK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value7>Q</span><span class=value3>S</span><span class=value6>SFDA</span><span class=value7>R</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value5>E</span><span class=value6>DSN</span><span class=value7>P</span><span class=value9>VK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value7>Q</span><span class=value3>S</span><span class=value6>SFDAR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value5>E</span><span class=value6>DSNP</span><span class=value9>VK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>A</span><span class=value6>TLDSR</span><span class=valuegap>--</span><span class=value5>S</span><span class=valuegap>-</span><span class=value4>A</span><span class=value6>SHRS</span><span class=value9>TR</span><span class=valuegap>----</span><span class=value8>F</span><span class=value9>AA</span><span class=valuegap>--------</span><span class=value9>AFY</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>A</span><span class=value6>TLDSR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value4>A</span><span class=value6>SHRS</span><span class=value9>TR</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AFY</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>A</span><span class=value6>ALDSR</span><span class=valuegap>--</span><span class=value6>S</span><span class=valuegap>-</span><span class=value4>A</span><span class=value6>SHRS</span><span class=value9>TR</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AFY</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>A</span><span class=value6>ALDSR</span><span class=valuegap>--</span><span class=value5>S</span><span class=valuegap>-</span><span class=value4>A</span><span class=value6>SHRS</span><span class=value9>TR</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AFY</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LVAIA</span><span class=valuegap>--</span><span class=value8>L</span><span class=value2>L</span><span class=valuegap>-</span><span class=value9>C</span><span class=value7>AQP</span><span class=value8>VL</span><span class=valuegap>----</span><span class=value1>T</span><span class=valuegap>----------</span><span class=value7>SEQ</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LVAIA</span><span class=valuegap>--</span><span class=value8>L</span><span class=value2>L</span><span class=valuegap>-</span><span class=value9>C</span><span class=value7>AQP</span><span class=value8>VL</span><span class=valuegap>----</span><span class=value1>T</span><span class=valuegap>----------</span><span class=value7>SEQ</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LVAIA</span><span class=valuegap>--</span><span class=value8>L</span><span class=value2>L</span><span class=valuegap>-</span><span class=value9>C</span><span class=value7>AQP</span><span class=value8>VL</span><span class=valuegap>----</span><span class=value1>T</span><span class=valuegap>----------</span><span class=value7>SEQ</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value7>F</span><span class=value6>G</span><span class=value7>GFL</span><span class=valuegap>--</span><span class=value7>V</span><span class=value2>A</span><span class=valuegap>-</span><span class=value7>LALRVV</span><span class=valuegap>----</span><span class=value1>G</span><span class=value7>AQ</span><span class=value2>R</span><span class=value3>QD</span><span class=value5>V</span><span class=value4>LLQ</span><span class=value2>N</span><span class=value7>STT</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHHAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value1>E</span><span class=valuegap>--------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHHAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value1>E</span><span class=valuegap>--------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>KKH</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHHAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value1>E</span><span class=valuegap>--------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value7>L</span><span class=value6>GGFL</span><span class=valuegap>--</span><span class=value6>V</span><span class=value1>A</span><span class=valuegap>-</span><span class=value6>LALQ</span><span class=value7>TV</span><span class=valuegap>----</span><span class=value1>G</span><span class=value7>AK</span><span class=value9>Q</span><span class=value3>ED</span><span class=value4>F</span><span class=value3>F</span><span class=value4>HR</span><span class=value2>N</span><span class=value7>STP</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>KKH</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FQNAK</span><span class=valuegap>--</span><span class=value8>F</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FQNAK</span><span class=valuegap>--</span><span class=value8>F</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LG</span><span class=value9>A</span><span class=value8>FL</span><span class=valuegap>--</span><span class=value7>V</span><span class=value2>A</span><span class=valuegap>-</span><span class=value8>LALQT</span><span class=value9>V</span><span class=valuegap>----</span><span class=value2>G</span><span class=value6>AK</span><span class=valuegap>-</span><span class=value9>ED</span><span class=value4>L</span><span class=value3>FP</span><span class=value4>R</span><span class=value8>N</span><span class=value7>STP</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>KKH</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FQNAK</span><span class=valuegap>--</span><span class=value8>F</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value7>F</span><span class=value6>H</span><span class=value7>N</span><span class=value6>AK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>SQAAPA</span><span class=value8>Q</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FQNAK</span><span class=valuegap>--</span><span class=value8>F</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value6>L</span><span class=valuegap>-------------------------------</span><span class=value6>EE</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAH</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAH</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FQNAK</span><span class=valuegap>--</span><span class=value8>F</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAH</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPTQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DG</span><span class=valuegap>--------</span><span class=value8>KKH</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>EE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>EE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>EE</span><span class=valuegap>--------</span><span class=value8>EKQ</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKK</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>DE</span><span class=valuegap>--------</span><span class=value8>EKK</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>FHNAK</span><span class=valuegap>--</span><span class=value8>L</span><span class=valuegap>-</span><span class=value7>S</span><span class=value8>QAAPAQ</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value8>NE</span><span class=valuegap>--------</span><span class=value8>EKE</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>I</span><span class=value7>L</span><span class=value8>YCA</span><span class=valuegap>--</span><span class=value8>A</span><span class=value1>T</span><span class=valuegap>-</span><span class=value8>VMRP</span><span class=value7>EA</span><span class=valuegap>-----</span><span class=value8>DA</span><span class=valuegap>--------</span><span class=value8>THP</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>L</span><span class=value7>L</span><span class=value8>SCA</span><span class=valuegap>--</span><span class=value8>A</span><span class=value1>D</span><span class=value5>T</span><span class=value6>MMSP</span><span class=value7>EA</span><span class=valuegap>-----</span><span class=value8>DA</span><span class=valuegap>--------</span><span class=value8>THP</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>K</span><span class=valuegap>---</span><span class=value8>PAWQL</span><span class=value6>L</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>KA</span><span class=valuegap>---</span><span class=value7>YIHG</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>K</span><span class=valuegap>---</span><span class=value8>PAWQL</span><span class=value6>L</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>KA</span><span class=valuegap>---</span><span class=value5>Y</span><span class=value3>I</span><span class=value4>H</span><span class=value5>G</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value7>LHYCA</span><span class=valuegap>--</span><span class=value7>A</span><span class=value1>T</span><span class=valuegap>-</span><span class=value7>TILPHP</span><span class=value1>T</span><span class=value0>N</span><span class=value1>E</span><span class=value3>M</span><span class=value1>E</span><span class=value6>PV</span><span class=valuegap>--------</span><span class=value7>VEP</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>R</span><span class=valuegap>---</span><span class=value8>PTWQL</span><span class=value6>L</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>KA</span><span class=valuegap>---</span><span class=value5>Y</span><span class=value3>I</span><span class=value4>H</span><span class=value5>G</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value5>S</span><span class=value3>Y</span><span class=value8>L</span><span class=value7>N</span><span class=value8>T</span><span class=value7>T</span><span class=value8>R</span><span class=valuegap>--</span><span class=value8>I</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PG</span><span class=value8>Q</span><span class=value5>NPV</span><span class=valuegap>---</span><span class=value4>S</span><span class=value3>F</span><span class=value5>LR</span><span class=valuegap>--------</span><span class=value5>GRR</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value5>S</span><span class=value3>Y</span><span class=value8>L</span><span class=value7>N</span><span class=value8>T</span><span class=value7>TR</span><span class=valuegap>--</span><span class=value8>I</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PG</span><span class=value8>Q</span><span class=value5>NPV</span><span class=valuegap>---</span><span class=value4>S</span><span class=value3>F</span><span class=value5>LR</span><span class=valuegap>--------</span><span class=value5>GRR</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value5>S</span><span class=value3>Y</span><span class=value8>L</span><span class=value7>N</span><span class=value8>T</span><span class=value7>TR</span><span class=valuegap>--</span><span class=value8>I</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PG</span><span class=value8>Q</span><span class=value5>NPV</span><span class=valuegap>---</span><span class=value4>S</span><span class=value3>F</span><span class=value5>LR</span><span class=valuegap>--------</span><span class=value5>GRR</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value5>S</span><span class=value3>Y</span><span class=value8>L</span><span class=value7>N</span><span class=value8>T</span><span class=value7>TR</span><span class=valuegap>--</span><span class=value8>I</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PG</span><span class=value8>Q</span><span class=value5>NPV</span><span class=valuegap>---</span><span class=value4>S</span><span class=value3>F</span><span class=value5>LR</span><span class=valuegap>--------</span><span class=value5>GRR</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value5>F</span><span class=value3>Y</span><span class=value8>LNS</span><span class=value7>T</span><span class=value8>R</span><span class=valuegap>--</span><span class=value8>I</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PGR</span><span class=value6>NHV</span><span class=valuegap>---</span><span class=value5>P</span><span class=value3>F</span><span class=value5>FR</span><span class=valuegap>--------</span><span class=value5>G</span><span class=value6>RR</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>T</span><span class=valuegap>---</span><span class=value8>DTAQY</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RN</span><span class=valuegap>---</span><span class=value4>F</span><span class=valuegap>-</span><span class=value4>YE</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>T</span><span class=valuegap>---</span><span class=value8>DTAQY</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RN</span><span class=valuegap>---</span><span class=value4>F</span><span class=valuegap>-</span><span class=value4>YE</span><span class=valuegap>-</span><span class=value6>KKS</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>T</span><span class=valuegap>---</span><span class=value8>DTAQY</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RN</span><span class=valuegap>---</span><span class=value4>F</span><span class=valuegap>-</span><span class=value4>YE</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>T</span><span class=valuegap>---</span><span class=value8>DTAQY</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RN</span><span class=valuegap>---</span><span class=value4>F</span><span class=valuegap>-</span><span class=value4>YE</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>F</span><span class=value3>Y</span><span class=value8>LNS</span><span class=value7>T</span><span class=value8>R</span><span class=valuegap>--</span><span class=value8>K</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>PGQ</span><span class=value5>N</span><span class=value6>YV</span><span class=valuegap>---</span><span class=value4>P</span><span class=value3>F</span><span class=value5>SR</span><span class=valuegap>--------</span><span class=value5>G</span><span class=value6>KR</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>E</span><span class=valuegap>---</span><span class=value8>TNL</span><span class=value1>I</span><span class=value2>PH</span><span class=valuegap>-</span><span class=value2>GVH</span><span class=valuegap>--</span><span class=value6>VI</span><span class=valuegap>---</span><span class=value5>K</span><span class=value4>H</span><span class=value6>TQ</span><span class=valuegap>--------</span><span class=value6>DNR</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>T</span><span class=valuegap>---</span><span class=value8>DTAPH</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RS</span><span class=valuegap>---</span><span class=value4>F</span><span class=valuegap>-</span><span class=value4>YE</span><span class=valuegap>-</span><span class=value8>RKS</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=value2>E</span><span class=valuegap>---</span><span class=value8>TNL</span><span class=valuegap>-</span><span class=value2>I</span><span class=value3>L</span><span class=valuegap>-</span><span class=value3>HGV</span><span class=value2>H</span><span class=valuegap>-</span><span class=value6>SI</span><span class=valuegap>---</span><span class=value5>K</span><span class=value4>H</span><span class=value6>AQ</span><span class=valuegap>--------</span><span class=value6>DNR</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=value5>E</span><span class=valuegap>---</span><span class=value7>TNL</span><span class=value5>I</span><span class=value6>P</span><span class=value7>H</span><span class=valuegap>-</span><span class=value7>SI</span><span class=value6>H</span><span class=valuegap>--</span><span class=value6>SV</span><span class=valuegap>---</span><span class=value6>KH</span><span class=value5>GQ</span><span class=valuegap>--------</span><span class=value6>ENR</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value3>A</span><span class=value6>NLNSR</span><span class=valuegap>--</span><span class=value6>T</span><span class=valuegap>-</span><span class=value5>E</span><span class=value6>ET</span><span class=valuegap>--</span><span class=value6>IK</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>AHY</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value2>T</span><span class=value6>SMDSR</span><span class=valuegap>--</span><span class=value5>S</span><span class=valuegap>-</span><span class=value4>A</span><span class=value6>SHRS</span><span class=value9>TR</span><span class=valuegap>----</span><span class=value9>FAA</span><span class=valuegap>--------</span><span class=value9>TFY</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LHHAK</span><span class=valuegap>--</span><span class=value8>W</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPMA</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value8>GG</span><span class=valuegap>--------</span><span class=value8>GQN</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LHHAK</span><span class=valuegap>--</span><span class=value8>W</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPMA</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value8>GG</span><span class=valuegap>--------</span><span class=value8>GQN</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>LHHAK</span><span class=valuegap>--</span><span class=value8>W</span><span class=valuegap>-</span><span class=value6>S</span><span class=value8>QAAPMA</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value8>GG</span><span class=valuegap>--------</span><span class=value8>GQN</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value7>LAPAQ</span><span class=valuegap>--</span><span class=value6>A</span><span class=value1>P</span><span class=valuegap>-</span><span class=value6>VSQPDA</span><span class=valuegap>-----</span><span class=value6>PG</span><span class=valuegap>--------</span><span class=value6>HQR</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value8>L</span><span class=value7>A</span><span class=value8>P</span><span class=value7>A</span><span class=value8>Q</span><span class=valuegap>--</span><span class=value6>A</span><span class=value1>P</span><span class=valuegap>-</span><span class=value6>VSQPD</span><span class=value7>A</span><span class=valuegap>-----</span><span class=value6>PG</span><span class=valuegap>--------</span><span class=value7>HQ</span><span class=value6>R</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>F</span><span class=value8>LQ</span><span class=value7>LL</span><span class=valuegap>--</span><span class=value6>A</span><span class=value1>G</span><span class=valuegap>-</span><span class=value6>LALP</span><span class=value7>AV</span><span class=valuegap>----</span><span class=value1>P</span><span class=value7>PQ</span><span class=valuegap>-</span><span class=value2>WA</span><span class=value3>L</span><span class=value2>S</span><span class=value3>AG</span><span class=value2>N</span><span class=value7>GSS</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value6>LLAVA</span><span class=valuegap>--</span><span class=value6>I</span><span class=value1>L</span><span class=valuegap>-</span><span class=value6>FCIQGW</span><span class=valuegap>------</span><span class=value0>P</span><span class=valuegap>--------</span><span class=value6>SGT</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>D</span><span class=valuegap>---</span><span class=value8>TSL</span><span class=valuegap>-</span><span class=value2>R</span><span class=value3>A</span><span class=valuegap>-</span><span class=value3>HGV</span><span class=value6>H</span><span class=valuegap>-</span><span class=value6>AT</span><span class=valuegap>---</span><span class=value5>K</span><span class=value4>H</span><span class=value6>VP</span><span class=valuegap>--------</span><span class=value6>EKR</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>LD</span><span class=value8>L</span><span class=value7>NMT</span><span class=value8>R</span><span class=valuegap>--</span><span class=value8>S</span><span class=valuegap>-</span><span class=value6>H</span><span class=value9>SGG</span><span class=value5>E</span><span class=value6>LE</span><span class=valuegap>---</span><span class=value4>S</span><span class=value3>L</span><span class=value5>AR</span><span class=valuegap>--------</span><span class=value5>GRR</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>R</span><span class=valuegap>---</span><span class=value7>PTW</span><span class=value8>QL</span><span class=value6>L</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>KA</span><span class=valuegap>---</span><span class=value5>F</span><span class=value3>V</span><span class=value4>F</span><span class=value5>G</span><span class=valuegap>-</span><span class=value7>R</span><span class=value8>K</span><span class=value7>S</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value2>L</span><span class=valuegap>---</span><span class=value7>DT</span><span class=value8>PRY</span><span class=value6>R</span><span class=valuegap>---</span><span class=value5>G</span><span class=valuegap>-</span><span class=value6>RS</span><span class=valuegap>---</span><span class=value4>Y</span><span class=valuegap>-</span><span class=value4>HD</span><span class=valuegap>-</span><span class=value8>RK</span><span class=value7>S</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value2>&nbsp;</span><span class=value8>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value1>&nbsp;</span><span class=value6>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value5>&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value2>&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value2>&nbsp;</span><span class=value4>&nbsp;&nbsp;</span><span class=value2>&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>P</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEY</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>P</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEY</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>A</span><span class=value3>D</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEF</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>A</span><span class=value3>D</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEF</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>P</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEY</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>P</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCVDVGKEY</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>D</span><span class=valuegap>-</span><span class=value9>I</span><span class=value3>E</span><span class=valuegap>--</span><span class=value4>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>VIDEEWQRTQCMPRETCVEVAKEL</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value9>D</span><span class=valuegap>-</span><span class=value9>I</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>VIDEEWQRTQCVPRETCVEVAKEL</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value9>D</span><span class=valuegap>-</span><span class=value9>I</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>VIDEEWQRTQCVPRETCVEVAKEL</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value9>D</span><span class=valuegap>-</span><span class=value9>I</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>VIDEEWQRTQCVPRETCVEVAKEL</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------------</span><span class=value9>TQCVPRETCVDVAKDL</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value7>P</span><span class=valuegap>-</span><span class=value7>A</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>PFEEVWSRSYCRARETLVDILSEF</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value7>P</span><span class=valuegap>-</span><span class=value7>A</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>PFEEVWSRSYCRARETLVDILSEF</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value7>P</span><span class=valuegap>-</span><span class=value7>A</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>PFEEVWSRSYCRARETLVDILSEF</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=value9>E</span><span class=valuegap>-</span><span class=value8>A</span><span class=valuegap>---</span><span class=value7>A</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>SFHDVWNRSSCQPLEKLVDVVSEY</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>S</span><span class=valuegap>-</span><span class=value8>G</span><span class=valuegap>---</span><span class=value8>Y</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFLEVYERSRCRPIETMVDISHEY</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>S</span><span class=valuegap>-</span><span class=value8>G</span><span class=valuegap>---</span><span class=value8>D</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFMEVFARSRCKPIETMVDIFQEY</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLEVFRRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFVDVYNRSVCRSIETMVDIFQEY</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>S</span><span class=valuegap>-</span><span class=value8>G</span><span class=valuegap>---</span><span class=value8>D</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFMEVFARSRCKPIETMVDIFQEY</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=value9>E</span><span class=valuegap>-</span><span class=value8>V</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value9>VL</span><span class=valuegap>--</span><span class=value9>SFYDVWNRSSCQPMEKLVNIVSEY</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFVDVYNRSVCRSIETMVDIFQEY</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=value9>Q</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLVVFERSACRTIETMVDIFQEY</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=value9>Q</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLVVFERSACRTIETMVDIFQEY</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>E</span><span class=valuegap>-</span><span class=value8>E</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value9>VL</span><span class=valuegap>--</span><span class=value9>SFYDVWNRSSCQPMEKLVNIVSEY</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFVDVYNRSVCRSIETMVDIFQEY</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>TFIEVFSRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=value9>Q</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLVVFERSACRTIETMVDIFQEY</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value7>VI</span><span class=valuegap>--</span><span class=value9>SFLEVFRRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=value9>Q</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLVVFERSACRTIETMVDIFQEY</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=value6>A</span><span class=valuegap>-</span><span class=value5>C</span><span class=value2>G</span><span class=value1>CL</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>AL</span><span class=valuegap>--</span><span class=value9>PFEVVWDLSFCASRVTLVDVLSEF</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFLEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFLEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>QFLEVFDRSRCRSIETMVDIFQEY</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=value9>Q</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>SFLVVFERSACRTIETMVDIFQEY</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFLEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>QFLDVYNRSMCRSIETMVDIFQEY</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFMEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFMEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFMEVFGRSGCRSIETMVDIFQEY</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFYEVFDRSSCRSIETMVDIFQEY</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFYEVFDRSSCRAIETMVDIFQEY</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>P</span><span class=valuegap>-</span><span class=value8>S</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>PFFEVFSRSGCRSIETLVDIFQEY</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=value8>S</span><span class=valuegap>-</span><span class=value8>G</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>RWMDVYNRSFCQPKETLVPVSAEH</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>S</span><span class=valuegap>-</span><span class=value8>G</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VI</span><span class=valuegap>--</span><span class=value9>RWMEVYNRSFCQPKETLVPVSAEH</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>R</span><span class=value2>V</span><span class=value8>V</span><span class=value2>D</span><span class=valuegap>----</span><span class=value9>LN</span><span class=valuegap>--</span><span class=value9>LLKEEVRMYSCTPRNFSVSLREEL</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value8>R</span><span class=value2>V</span><span class=value6>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LN</span><span class=valuegap>--</span><span class=value9>LLKEEVRMYSCTPRNFSVSLREEL</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=value9>V</span><span class=valuegap>-</span><span class=value8>T</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>RFLEVFSRSACQAKETMVSLDKEY</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value8>R</span><span class=value2>V</span><span class=value6>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LN</span><span class=valuegap>--</span><span class=value9>LLKEEVRMYSCTPRNYSVSLREEL</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>L</span><span class=valuegap>------</span><span class=value3>V</span><span class=value2>GGE</span><span class=value6>PV</span><span class=valuegap>-</span><span class=value9>EPAVIAECKTRTEVFEISRTM</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>L</span><span class=valuegap>------</span><span class=value2>VGGE</span><span class=value6>PV</span><span class=valuegap>-</span><span class=value9>EPAVIAECKTRTEVFEISRTM</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>L</span><span class=valuegap>------</span><span class=value3>V</span><span class=value2>GGE</span><span class=value6>PV</span><span class=valuegap>-</span><span class=value9>EPAVIAECKTRTEVFEISRTM</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>L</span><span class=valuegap>------</span><span class=value3>V</span><span class=value2>GGE</span><span class=value7>PV</span><span class=valuegap>-</span><span class=value9>EPAVIAECKTRTEVFEISRTM</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>I</span><span class=valuegap>-</span><span class=value3>V</span><span class=valuegap>----</span><span class=value6>SG</span><span class=value5>GE</span><span class=value9>EPIEAAVIAECKTRTEVFEISRSM</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value8>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDVKRYSCTPRNYSVNIREEL</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDVKRYSCTPRNYSVNIREEL</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value8>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDVKRYSCTPRNYSVNIREEL</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value8>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDVKRYSCTPRNYSVNIREEL</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=value2>I</span><span class=valuegap>-</span><span class=value6>V</span><span class=valuegap>----</span><span class=value6>SG</span><span class=value5>GE</span><span class=value9>EPIEPAVIAECKTRAEVFEISRSM</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>P</span><span class=value2>R</span><span class=valuegap>-</span><span class=value2>P</span><span class=valuegap>----</span><span class=value6>IR</span><span class=value5>RK</span><span class=value9>RSIEEAIPAVCKTRTVIYEIPRSQ</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value8>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDVKRYSCTPRNYSVNIREEL</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=value6>P</span><span class=value2>R</span><span class=valuegap>-</span><span class=value2>P</span><span class=valuegap>----</span><span class=value6>VR</span><span class=value5>RK</span><span class=value9>RSIEEAIPAVCKTRTVIYEIPRSQ</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=value6>P</span><span class=value5>R</span><span class=valuegap>-</span><span class=value5>P</span><span class=valuegap>----</span><span class=value6>I</span><span class=value5>R</span><span class=value6>RK</span><span class=value9>RSVEEAIPAVCKTRTVIYEIPRSQ</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><span class=valuegap>-</span><span class=value9>T</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>I</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>SIDNEWRKTQCMPREVCIDVGKEF</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>D</span><span class=valuegap>-</span><span class=value9>I</span><span class=value3>E</span><span class=valuegap>--</span><span class=value6>T</span><span class=valuegap>-</span><span class=value9>LK</span><span class=valuegap>--</span><span class=value9>VIDEEWQRTQCSPRETCVEVASEL</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>H</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>KFMDVYQRSYCHPIETLVDIFQEY</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>H</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>KFMDVYQRSYCHPIETLVDIFQEY</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>H</span><span class=valuegap>-</span><span class=value8>H</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>KFMDVYQRSYCHPIETLVDIFQEY</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value7>K</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>SWIDVYTRATCQPREVVVPLTVEL</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value7>K</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>SWIDVYTRATCQPREVVVPLTVEL</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>E</span><span class=valuegap>-</span><span class=value8>V</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value9>VV</span><span class=valuegap>--</span><span class=value9>PFQEVWGRSYCRALERLVDVVSEY</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>V</span><span class=valuegap>-</span><span class=value5>Q</span><span class=valuegap>---</span><span class=value4>G</span><span class=value1>Q</span><span class=value9>VR</span><span class=valuegap>--</span><span class=value9>PFLEVHERSACQARETLVPILQEY</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>P</span><span class=value1>L</span><span class=valuegap>-</span><span class=value2>P</span><span class=valuegap>----</span><span class=value6>IR</span><span class=value5>RK</span><span class=value9>RSIEEAVPAVCKTRTVIYEIPRSQ</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>S</span><span class=valuegap>-------</span><span class=value1>L</span><span class=value2>G</span><span class=value1>SL</span><span class=value9>TIAEPAMIAECKTRTEVFEISRRL</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>R</span><span class=value2>V</span><span class=value6>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LN</span><span class=valuegap>--</span><span class=value9>LLTEEVRLYSCTPRNFSVSIREEL</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>K</span><span class=valuegap>-</span><span class=value7>V</span><span class=value3>D</span><span class=valuegap>----</span><span class=value9>LD</span><span class=valuegap>--</span><span class=value9>RLNDDAKRYSCTPRNYSVNIREEL</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;</span><span class=value1>&nbsp;</span><span class=value8>&nbsp;</span><span class=value2>&nbsp;</span><span class=value0>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value0>&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><span class=value3>&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GA</span><span class=valuegap>-</span><span class=value9>TTNTFFKPPCVSIYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GT</span><span class=valuegap>-</span><span class=value9>TTNKFFKPPCVNVYRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESLS</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GT</span><span class=valuegap>-</span><span class=value9>TTNKFFKPPCVNVYRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESLS</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GT</span><span class=valuegap>-</span><span class=value9>ATNKFFKPPCVNVYRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESLS</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GT</span><span class=valuegap>-</span><span class=value9>ATNKFFKPPCVNVYRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESLS</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------------------</span><span class=value9>NS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GT</span><span class=valuegap>-</span><span class=value9>TTNKFFKPPCVNVYRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESRS</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PH</span><span class=valuegap>-</span><span class=value9>EAEHIFKPPCVPLWRCAGCCGD</span><span class=valuegap>-----</span><span class=value9>ESLE</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PH</span><span class=valuegap>-</span><span class=value9>EAEHIFKPPCVPLWRCAGCCGD</span><span class=valuegap>-----</span><span class=value9>ESLE</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PH</span><span class=valuegap>-</span><span class=value9>EAEHIFKPPCVPLWRCAGCCGD</span><span class=valuegap>-----</span><span class=value9>ESLE</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PS</span><span class=valuegap>-</span><span class=value9>EVEHVFSPSCVFLQRCSGCCGD</span><span class=valuegap>-----</span><span class=value9>ERLQ</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PE</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVSLMRCGGCCSD</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PE</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYFFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PE</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCARCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PC</span><span class=valuegap>-</span><span class=value9>EVEHMFSPSCVSLHRCSGCCGD</span><span class=valuegap>-----</span><span class=value9>ETLH</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYFFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PC</span><span class=valuegap>-</span><span class=value9>EVEHMFSPSCVSLHRCSGCCGD</span><span class=valuegap>-----</span><span class=value9>ETLQ</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYFFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PH</span><span class=valuegap>-</span><span class=value9>DVEXIYKPLCVPLSRCNGCCND</span><span class=valuegap>-----</span><span class=value9>ERLE</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLMRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFMPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EVEYIFKPSCVPLLRCAGCCND</span><span class=valuegap>-----</span><span class=value9>EALE</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PG</span><span class=valuegap>-</span><span class=value9>EVEHILVPSCVPLRRCVGCCSD</span><span class=valuegap>-----</span><span class=value9>EELQ</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PG</span><span class=valuegap>-</span><span class=value9>EVEHILVPSCVPLRRCVGCCSD</span><span class=valuegap>-----</span><span class=value9>EELQ</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value4>K</span><span class=valuegap>-</span><span class=value9>RTDTIFWPLCLLVKRCGGNCA</span><span class=value6>C</span><span class=value7>CQHSC</span><span class=value6>SE</span><span class=value9>CQ</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value4>K</span><span class=valuegap>-</span><span class=value9>RTDTIFWPLCLLVKRCGGNCA</span><span class=value6>C</span><span class=value3>CQHSC</span><span class=value6>SE</span><span class=value9>CQ</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PDLVISHLIVPSCVALKRCVGCCAD</span><span class=valuegap>-----</span><span class=value9>ESAQ</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value4>K</span><span class=valuegap>-</span><span class=value9>RTDTIFWPLCLLVKRCGGNCG</span><span class=value6>C</span><span class=value3>CQHAC</span><span class=value6>SE</span><span class=value9>CQ</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFLVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMQ</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFLVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMQ</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFLVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMQ</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFLVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMQ</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFIVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMH</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>R</span><span class=valuegap>-</span><span class=value9>VTNAVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GFTHW</span><span class=value6>KS</span><span class=value9>CT</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>R</span><span class=valuegap>-</span><span class=value9>VTNAVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GFTHW</span><span class=value6>KS</span><span class=value9>CT</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>R</span><span class=valuegap>-</span><span class=value9>VTNAVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GFTHW</span><span class=value6>KS</span><span class=value9>CT</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>R</span><span class=valuegap>-</span><span class=value9>VTNAVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GFTHW</span><span class=value6>KS</span><span class=value9>CT</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>NANFIVWPPCVEVQRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>RNMH</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>IDPT</span><span class=valuegap>-</span><span class=value9>SANFLIWPPCVEVKRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>SSVK</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>K</span><span class=valuegap>-</span><span class=value9>QTNVVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GFNNW</span><span class=value6>KS</span><span class=value9>CT</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=value9>IDPT</span><span class=valuegap>-</span><span class=value9>SANFLIWPPCVEVKRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>SSVK</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=value9>IDPT</span><span class=valuegap>-</span><span class=value9>SANFLIWPPCVEVKRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>SSVK</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GV</span><span class=valuegap>-</span><span class=value9>ATNTFFKPPCVSVYRCGGCCNS</span><span class=valuegap>-----</span><span class=value9>EGLQ</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>GK</span><span class=valuegap>-</span><span class=value9>STNTFFKPPCVNVFRCGGCCNE</span><span class=valuegap>-----</span><span class=value9>ESLI</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EIEYIFKPSCVPLMRCGGCCND</span><span class=valuegap>-----</span><span class=value9>EGLE</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EIEYIFKPSCVPLMRCGGCCND</span><span class=valuegap>-----</span><span class=value9>EGLE</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EIEYIFKPSCVPLMRCGGCCND</span><span class=valuegap>-----</span><span class=value9>EGLE</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>MG</span><span class=valuegap>-</span><span class=value9>TVAKQLVPSCVTVQRCGGCCPD</span><span class=valuegap>-----</span><span class=value9>DGLE</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>MG</span><span class=valuegap>-</span><span class=value9>TVAKQLVPSCVTVQRCGGCCPD</span><span class=valuegap>-----</span><span class=value9>DGLE</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PS</span><span class=valuegap>-</span><span class=value9>EVEHMFSPSCVSLLRCTGCCGD</span><span class=valuegap>-----</span><span class=value9>ENLH</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PD</span><span class=valuegap>-</span><span class=value9>EISDIFRPSCVAVLRCSGCCTD</span><span class=valuegap>-----</span><span class=value9>ESLK</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>VDPT</span><span class=valuegap>-</span><span class=value9>SANFLIWPPCVEVKRCTGCCNT</span><span class=valuegap>-----</span><span class=value9>SSVK</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>IDRT</span><span class=valuegap>-</span><span class=value9>NANFLVWPPCVEVQRCSGCCNN</span><span class=valuegap>-----</span><span class=value9>RNVQ</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>K</span><span class=valuegap>-</span><span class=value9>RTDTIFWPGCLLVKRCGGNCA</span><span class=value6>C</span><span class=value3>CLHNC</span><span class=value6>NE</span><span class=value9>CQ</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---</span><span class=value5>K</span><span class=valuegap>-</span><span class=value9>LANVVFFPRCLLVQRCGGNCG</span><span class=value6>C</span><span class=value3>GTVNW</span><span class=value6>RS</span><span class=value9>CT</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSFISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSFISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSYISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSYISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSFISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSFISKTLFEITVP</span><span class=value2>LS</span><span class=value8>HGP</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CTNTSTSYVSKMLFEISVP</span><span class=value2>LT</span><span class=value7>N</span><span class=value8>V</span><span class=value7>P</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNTSTSYVSKMLFEISVP</span><span class=value2>LT</span><span class=value8>NV</span><span class=value7>P</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVNTSTSYISKMLFEISVP</span><span class=value2>LT</span><span class=value8>NV</span><span class=value7>P</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVNTSTSYISKMLFEISVP</span><span class=value2>LT</span><span class=value8>NV</span><span class=value7>P</span><span class=valuegap>------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNISTSYISKTLFEITVP</span><span class=value2>LS</span><span class=value9>HG</span><span class=value8>P</span><span class=valuegap>------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNT</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CIAVETRTIDLQVVRVS</span><span class=value8>P</span><span class=value9>I</span><span class=valuegap>--</span><span class=value9>LG</span><span class=value8>T</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CIAVETRTIDLQVVRVS</span><span class=value8>P</span><span class=value9>I</span><span class=valuegap>--</span><span class=value9>LG</span><span class=value8>T</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CIAVETRTIDLQVVRVS</span><span class=value8>P</span><span class=value9>I</span><span class=valuegap>--</span><span class=value9>LG</span><span class=value8>T</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVANVTMQLLKVN</span><span class=value8>S</span><span class=value9>E</span><span class=valuegap>--</span><span class=value9>EQ</span><span class=value8>A</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPLELQNVTMEVMRIKIH</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMPLEVENVTMEVMRIKFH</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEILRFKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVALEVHNVTMEIMRIKPY</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMPLEVENVTMEVMRIKFH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEMTNVTMQLLRMK</span><span class=value8>S</span><span class=value9>G</span><span class=valuegap>--</span><span class=value9>EQ</span><span class=value8>A</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVALEVHNVTMEIMRIKPY</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTSTYNVTMEILRIKHF</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTSTYNVTMEILRIKHF</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEMTNVTMQLLRMK</span><span class=value8>S</span><span class=value9>G</span><span class=valuegap>--</span><span class=value9>NQ</span><span class=value8>A</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVALEVHNVTMEIMRIKPY</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEILRIKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTSTYNVTMEILRIKHF</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEILRFKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value7>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTSTYNVTMEILRIKHF</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CAPVETRPLDLQVMRIS</span><span class=value8>V</span><span class=value9>L</span><span class=valuegap>--</span><span class=value9>QR</span><span class=value8>T</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPIEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTSTYNVTMEILRIKHF</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVALEVHNVTMEIMRIKPL</span><span class=valuegap>--</span><span class=value9>VS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRLKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRLKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEVHNVTMEIMRLKYH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPIEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPIEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPIEVHNVTMEIMRIKYH</span><span class=valuegap>--</span><span class=value9>QS</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPVQMHVVIMEVMKT</span><span class=value3>R</span><span class=valuegap>-</span><span class=value6>F</span><span class=valuegap>--</span><span class=value7>PR</span><span class=value8>S</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPVQMHVVIMEVMKT</span><span class=value4>R</span><span class=valuegap>-</span><span class=value6>F</span><span class=valuegap>--</span><span class=value7>PH</span><span class=value8>S</span><span class=valuegap>------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTKVTKKYHEVLQLK</span><span class=value8>P</span><span class=value9>R</span><span class=valuegap>--</span><span class=value9>SGV</span><span class=value7>RGL</span><span class=valuegap>-----</span><span class=value8>H</span><span class=value9>K</span><span class=valuegap>--</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CIPTKVTKKYHEVLQLK</span><span class=value8>PR</span><span class=valuegap>--</span><span class=value9>SG</span><span class=value6>VR</span><span class=value7>G</span><span class=value6>L</span><span class=valuegap>-----</span><span class=value8>H</span><span class=value4>K</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CMPSRTYTVTMEVLLLS</span><span class=value8>K</span><span class=value9>F</span><span class=valuegap>--</span><span class=value9>QR</span><span class=value8>S</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTKVTKKYHEVLQLK</span><span class=value8>PR</span><span class=valuegap>--</span><span class=value9>SG</span><span class=value6>VKGL</span><span class=valuegap>-----</span><span class=value8>H</span><span class=value4>K</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CRATRVRERHIQVNKIEIV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>V</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CRATRVRERHIQVNKIEIV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>V</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CRATRVRERHIQVNKIEIV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>V</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CRATRVRERHIQVNKIEIV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>V</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CLPTRVRVRHIQVNKIG</span><span class=value8>I</span><span class=value9>V</span><span class=valuegap>--</span><span class=value9>QK</span><span class=value7>KP</span><span class=value8>T</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CTPGKLVKKYHEVLRFI</span><span class=value8>PE</span><span class=valuegap>--</span><span class=value9>AG</span><span class=value8>HI</span><span class=value9>R</span><span class=value7>R</span><span class=value5>R</span><span class=value7>GR</span><span class=valuegap>--</span><span class=value9>TKNN</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CTPGKLVKKYHEVLRFI</span><span class=value8>PE</span><span class=valuegap>--</span><span class=value9>AGHIR</span><span class=value8>R</span><span class=value5>R</span><span class=value7>GR</span><span class=valuegap>--</span><span class=value9>TKNN</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CTPGKLVKKYHEVLRFI</span><span class=value8>PE</span><span class=valuegap>--</span><span class=value9>AG</span><span class=value8>HI</span><span class=value9>R</span><span class=value7>R</span><span class=value5>R</span><span class=value7>GR</span><span class=valuegap>--</span><span class=value9>TKNN</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CTPGKLVKKYHEVLRFI</span><span class=value8>PE</span><span class=valuegap>--</span><span class=value9>AG</span><span class=value8>HI</span><span class=value9>R</span><span class=value7>R</span><span class=value5>R</span><span class=value7>GR</span><span class=valuegap>--</span><span class=value9>TKNN</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CLPTRVRVRHIQVNKIRFV</span><span class=valuegap>--</span><span class=value9>QK</span><span class=value7>K</span><span class=value6>A</span><span class=value8>T</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CQPSRIHHRSVKVAKVEYV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>K</span><span class=valuegap>-------</span><span class=value2>L</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CTAGKLVKKYHEVLRFI</span><span class=value8>PE</span><span class=valuegap>--</span><span class=value9>AG</span><span class=value8>HI</span><span class=value9>R</span><span class=value7>R</span><span class=value5>R</span><span class=value7>GR</span><span class=valuegap>--</span><span class=value9>TKNN</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CQPSRIHHRSVKRA</span><span class=valuegap>----</span><span class=value6>F</span><span class=valuegap>--</span><span class=value6>GS</span><span class=value4>KIL</span><span class=valuegap>-------</span><span class=value1>N</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=value9>CQPSRIHHRNVKVRRNGTQ</span><span class=valuegap>--</span><span class=value9>VQ</span><span class=value7>RC</span><span class=value8>G</span><span class=value6>R</span><span class=valuegap>-----</span><span class=value8>V</span><span class=value5>G</span><span class=valuegap>--</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNTSTSYLSKTLFEITVP</span><span class=value2>LS</span><span class=value8>QGP</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CMNTSTSYISKQLFEISVP</span><span class=value2>LT</span><span class=value7>S</span><span class=value8>V</span><span class=value7>P</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEESNITMQIMRIKPH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEESNITMQIMRIKPH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTEESNITMQIMRIKPH</span><span class=valuegap>--</span><span class=value9>QG</span><span class=value8>Q</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTGQHQVRMQILMI</span><span class=value4>R</span><span class=valuegap>-</span><span class=value5>Y</span><span class=valuegap>--</span><span class=value6>PS</span><span class=value8>S</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPTGQHQVRMQILMI</span><span class=value4>R</span><span class=valuegap>-</span><span class=value5>Y</span><span class=valuegap>--</span><span class=value6>PS</span><span class=value8>S</span><span class=valuegap>------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPVETANVTMQLLKIR</span><span class=value8>S</span><span class=value9>G</span><span class=valuegap>--</span><span class=value9>DR</span><span class=value8>P</span><span class=valuegap>------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CTPVGKHTVDLQIMRVN</span><span class=value4>P</span><span class=value5>R</span><span class=valuegap>--</span><span class=value5>T</span><span class=value7>QS</span><span class=valuegap>------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CQPSRVHHRSVKVAKVEYV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>KP</span><span class=value8>K</span><span class=valuegap>------</span><span class=value3>L</span><span class=valuegap>---</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CRPTQVQLRPVQVRKIEIV</span><span class=valuegap>--</span><span class=value9>RK</span><span class=value7>K</span><span class=value6>P</span><span class=value8>I</span><span class=valuegap>------</span><span class=value3>F</span><span class=valuegap>---</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CVPSKVTKKYHEVLQLR</span><span class=value8>PK</span><span class=valuegap>--</span><span class=value9>TG</span><span class=value6>VRGL</span><span class=valuegap>-----</span><span class=value8>H</span><span class=value4>K</span><span class=valuegap>--</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>CNSGKTVKKYHEVLQFE</span><span class=value8>P</span><span class=valuegap>----</span><span class=value5>G</span><span class=value3>HI</span><span class=value5>K</span><span class=value4>RR</span><span class=value5>GRA</span><span class=value6>K</span><span class=value8>T</span><span class=valuegap>---</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>*&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><span class=value1>&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><span class=value8>&nbsp;</span><span class=value6>&nbsp;</span><span class=value7>&nbsp;</span><span class=value5>&nbsp;</span><span class=value4>&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value8>SKLD</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>DPIQVKIANHTGCKCI</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>NVQ</span><span class=valuegap>----</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>DPIQVKIANHTDCKCI</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>NGQ</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>DPIQVKIANHTGCKCL</span><span class=valuegap>-----------</span><span class=value7>T</span><span class=value8>NAQ</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>DPIQVKIANHTGCKCL</span><span class=valuegap>-----------</span><span class=value7>T</span><span class=value8>NAQ</span><span class=valuegap>----</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value8>S</span><span class=valuegap>-------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value3>T</span><span class=value9>QQEEMKFVEHTRCSC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PRRK</span><span class=valuegap>-</span><span class=value6>RLK</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value3>T</span><span class=value9>QQEEMKFVEHTRCSC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PRRK</span><span class=valuegap>-</span><span class=value6>RLK</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value3>T</span><span class=value9>QQEEMKFVEHTRCSC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PRRK</span><span class=valuegap>-</span><span class=value6>RLK</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>PYVELSFVEHRKCVC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PRRDAVKS</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HITRMTFQQHKKCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRHI</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HVSRMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRAI</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKACECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRVI</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HINLMSFQHHSKCGCR</span><span class=valuegap>-----------</span><span class=value8>PKKETRTR</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HVSRMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRAI</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SYVEMSFAEHKKCLC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>LRQEVPKP</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HINLMSFQHHSKCGCR</span><span class=valuegap>-----------</span><span class=value8>PKKETRTR</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHRECECR</span><span class=valuegap>---------</span><span class=value9>ER</span><span class=value8>PKKEVKPR</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHRECECR</span><span class=valuegap>---------</span><span class=value9>ER</span><span class=value8>PKKEVKPR</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SYVEMSFAEHKKCLC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>LRQEILKP</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HINLMSFQHHSKCGCR</span><span class=valuegap>-----------</span><span class=value8>PKKETRTR</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHKTCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>CEK</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHRECECR</span><span class=valuegap>---------</span><span class=value9>ER</span><span class=value8>PKKEVKPR</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKACEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>PKKE</span><span class=value8>TRVI</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHRECECR</span><span class=valuegap>---------</span><span class=value9>ER</span><span class=value8>PKKEVKPR</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=value3>A</span><span class=value9>EQVVLTFMEHMRCTC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>R</span><span class=value7>REM</span><span class=valuegap>-</span><span class=value6>RWK</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKTCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>CEK</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRVI</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HINQMSFQQHKTCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>CEK</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSRMSFQQHRECECR</span><span class=valuegap>---------</span><span class=value9>ER</span><span class=value8>PKKEVKPR</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRVI</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HINLMSFQHHSKCGCR</span><span class=valuegap>-----------</span><span class=value8>PKKETRTK</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSWMSFQQHKTCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>CEK</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSWMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRVI</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLSWMSFQQHKTCECR</span><span class=valuegap>-----------</span><span class=value8>PKKETRVI</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKKCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value7>CEK</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKKCEC</span><span class=value7>T</span><span class=valuegap>-----------</span><span class=value6>CE</span><span class=value7>K</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HISQMSFQQHKTCEC</span><span class=value7>R</span><span class=valuegap>-----------</span><span class=value6>CE</span><span class=value7>K</span><span class=value6>P</span><span class=valuegap>----</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLGQMSFVEHSACECR</span><span class=valuegap>-------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLGQMSFVEHSACEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PKKLIADK</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SLTDVPLEHHEECEC</span><span class=value7>I</span><span class=valuegap>-------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SLTDVPLEHHEECEC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HLQQLTFEEHATCEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PKEV</span><span class=value7>YRLA</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SLTDVPLDHHEECDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value8>A</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value8>A</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value8>A</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value8>A</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value8>A</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=value1>M</span><span class=value9>ALIDIQLEHHERCDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value1>M</span><span class=value9>ALIDIQLEHHERCDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=value1>M</span><span class=value9>ALIDIQLEHHERCDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=value1>M</span><span class=value9>ALIDIQLEHHERCDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>NKAIVALEDHLECRCE</span><span class=valuegap>---------</span><span class=value6>A</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KEVLVKLEEHMECTC</span><span class=value3>T</span><span class=value4>S</span><span class=value3>SSSN</span><span class=value4>S</span><span class=value5>D</span><span class=value4>YRE</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=value1>M</span><span class=value9>ALIDIQLEHHERCDC</span><span class=value7>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>GDILL</span><span class=value1>C</span><span class=value0>Q</span><span class=value1>S</span><span class=value0>KLIV</span><span class=value1>SVY</span><span class=valuegap>---------</span><span class=value1>RQ</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>VLREEGVQHFVRILS</span><span class=valuegap>---------</span><span class=value6>F</span><span class=value8>S</span><span class=valuegap>---------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KPVTISFANHTSCRCM</span><span class=valuegap>-----------</span><span class=value7>S</span><span class=value8>KLD</span><span class=valuegap>----</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>ELVPVKVANHTGCKCL</span><span class=valuegap>-----------</span><span class=value7>P</span><span class=value8>TAP</span><span class=valuegap>----</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HIGEMSFLQHNKCECR</span><span class=valuegap>-----------</span><span class=value8>PKKD</span><span class=valuegap>-</span><span class=value8>RAR</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HIGEMSFLQHNKCECR</span><span class=valuegap>-----------</span><span class=value8>PKKD</span><span class=valuegap>-</span><span class=value7>RAR</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>HIGEMSFLQHNKCECR</span><span class=valuegap>-----------</span><span class=value8>PKKD</span><span class=valuegap>-</span><span class=value7>RAR</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>QLGEMSLEEHSQCEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>P</span><span class=value7>KKKDSAV</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>QLGEMSLEEHSQCEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PKKK</span><span class=value7>DS</span><span class=value8>AV</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SYVELTFSQHVRCEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PLREKMKP</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>S</span><span class=value9>KMEVMKFTEHTACEC</span><span class=value8>R</span><span class=valuegap>-----------</span><span class=value8>PRRK</span><span class=value2>Q</span><span class=valuegap>---</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KEVQVRLEEHLECACA</span><span class=valuegap>---------</span><span class=value8>T</span><span class=valuegap>---------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>KKATVTLEDHLACKCE</span><span class=valuegap>---------</span><span class=value8>T</span><span class=valuegap>---------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value9>SLTDVALEHHEECDC</span><span class=value6>V</span><span class=valuegap>-------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>M</span><span class=value9>ALVDIQLDHHERCDC</span><span class=value6>I</span><span class=valuegap>-------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value1>&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value3>&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value9>&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value8>&nbsp;</span><span class=value7>&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value7>VY</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value5>G</span><span class=value6>E</span><span class=value5>S</span><span class=value2>S</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value5>GER</span><span class=value2>S</span><span class=valuegap>----</span><span class=value5>HRIGAKRRPREPSLTPF</span><span class=value4>T</span><span class=valuegap>----------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value5>GER</span><span class=value2>S</span><span class=valuegap>----</span><span class=value5>HRIGAKRRPREPSLTPF</span><span class=value4>T</span><span class=valuegap>----------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=value7>GRR</span><span class=value6>RRPKG</span><span class=value7>R</span><span class=valuegap>---</span><span class=value5>SKRKR</span><span class=value6>AKQ</span><span class=value4>RPK</span><span class=value5>D</span><span class=valuegap>------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>KE</span><span class=value7>KKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKRGRYKP</span><span class=value7>QT</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEN</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEL</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=value7>QE</span><span class=value6>K</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEN</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=value7>GRR</span><span class=value6>RRPKG</span><span class=value7>K</span><span class=value6>GRGS</span><span class=value7>KMK</span><span class=value8>R</span><span class=value5>ARHRLK</span><span class=value4>D</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=value7>QEN</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=value7>Q</span><span class=value8>E</span><span class=value7>KKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKRGRY</span><span class=value5>K</span><span class=valuegap>-------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=value8>QE</span><span class=value7>N</span><span class=value2>H</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>GRR</span><span class=value6>RRPKG</span><span class=value7>R</span><span class=value6>GRGS</span><span class=value7>KMKR</span><span class=value5>ARHRLK</span><span class=value4>D</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=value7>QEKKSKR</span><span class=value6>G</span><span class=value7>RGKGQKR</span><span class=value8>KRKRGRYKP</span><span class=value7>QN</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=value8>QE</span><span class=value7>KKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKRGR</span><span class=value2>H</span><span class=valuegap>--------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEK</span><span class=value8>KSKR</span><span class=value7>G</span><span class=value8>K</span><span class=value6>GKGQKR</span><span class=value8>K</span><span class=value7>RKKS</span><span class=value8>R</span><span class=value7>Y</span><span class=value8>K</span><span class=value7>H</span><span class=value8>QTL</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=value8>QE</span><span class=value7>KKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKRGRYKP</span><span class=value7>QN</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=value5>DER</span><span class=value2>H</span><span class=valuegap>----</span><span class=value5>RRTGAKRRPRKPSP</span><span class=value6>A</span><span class=value4>P</span><span class=value3>LA</span><span class=valuegap>----------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEN</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=value7>QEKKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KR</span><span class=value6>KR</span><span class=valuegap>---</span><span class=value6>V</span><span class=valuegap>-------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEKKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKKSRYKH</span><span class=value7>QT</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LEKKSKR</span><span class=value6>G</span><span class=value7>RGKGQKR</span><span class=value8>KRKRGRYKP</span><span class=value7>QN</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEN</span><span class=valuegap>---------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KEKKSKR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKKSRYKH</span><span class=value7>QT</span><span class=value6>F</span><span class=valuegap>---------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>TER</span><span class=valuegap>----------</span><span class=value2>LP</span><span class=value1>C</span><span class=valuegap>--------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=value7>THS</span><span class=valuegap>----------</span><span class=value2>PS</span><span class=value1>C</span><span class=valuegap>--------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------------------------</span><span class=value8>V</span><span class=value9>Y</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>QEKKSVR</span><span class=value6>G</span><span class=value7>KGKGQKR</span><span class=value8>KRKKSRYK</span><span class=value7>SWS</span><span class=value4>VYV</span><span class=value3>GA</span><span class=value5>RCCLM</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>QE</span><span class=valuegap>----------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>Q</span><span class=value7>E</span><span class=valuegap>----------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>KPD</span><span class=valuegap>----------</span><span class=value1>RAATP</span><span class=valuegap>------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>KPD</span><span class=valuegap>----------</span><span class=value2>SPRPL</span><span class=valuegap>------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value6>ER</span><span class=value5>RRPKG</span><span class=value6>R</span><span class=value5>G</span><span class=valuegap>---</span><span class=value5>KRRREKQRPTD</span><span class=value4>CH</span><span class=valuegap>----------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>GEP</span><span class=valuegap>-------</span><span class=value1>D</span><span class=value4>GP</span><span class=value5>KEKPR</span><span class=valuegap>------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value6>&nbsp;</span><span class=value5>&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value7>&nbsp;</span><span class=value8>&nbsp;</span><span class=value1>&nbsp;&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value1>&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLSVT</span><span class=value3>QT</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLSVT</span><span class=value3>QT</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value5>R</span><span class=valuegap>---</span><span class=value5>Q</span><span class=valuegap>-</span><span class=value5>V</span><span class=value6>HS</span><span class=valuegap>--</span><span class=value6>IIRRS</span><span class=value5>L</span><span class=value6>PLT</span><span class=value3>QQ</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLPLT</span><span class=value3>QP</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLSVT</span><span class=value3>QT</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLSVT</span><span class=value3>QT</span><span class=valuegap>-</span><span class=value4>QCHV</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>H</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>SYS</span><span class=valuegap>--</span><span class=value6>IIRRSVPYS</span><span class=value4>EE</span><span class=value3>D</span><span class=value4>RC</span><span class=value3>SH</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>H</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>PYS</span><span class=valuegap>--</span><span class=value6>IIRRSVPYS</span><span class=value4>EE</span><span class=value3>D</span><span class=value4>RCSH</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>R</span><span class=valuegap>---</span><span class=value6>P</span><span class=valuegap>-</span><span class=value6>SYT</span><span class=valuegap>--</span><span class=value6>IIRRSVQYP</span><span class=value4>EE</span><span class=value3>D</span><span class=value4>SC</span><span class=value3>PQ</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>R</span><span class=valuegap>---</span><span class=value6>P</span><span class=valuegap>-</span><span class=value6>SYT</span><span class=valuegap>--</span><span class=value6>IIRRSVQYP</span><span class=value4>EE</span><span class=value3>D</span><span class=value4>SCPQ</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>C</span><span class=valuegap>---</span><span class=value6>K</span><span class=valuegap>-</span><span class=value6>LP</span><span class=value7>G</span><span class=valuegap>--</span><span class=value7>AGR</span><span class=value6>P</span><span class=value8>E</span><span class=value6>AQHR</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PL</span><span class=valuegap>-----</span><span class=value6>C</span><span class=valuegap>---</span><span class=value6>K</span><span class=valuegap>-</span><span class=value6>LPG</span><span class=valuegap>--</span><span class=value6>AG</span><span class=value7>R</span><span class=value6>P</span><span class=value8>E</span><span class=value6>AQHR</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PLS</span><span class=valuegap>----</span><span class=value6>C</span><span class=valuegap>---</span><span class=value6>K</span><span class=valuegap>-</span><span class=value6>LPG</span><span class=valuegap>--</span><span class=value6>AG</span><span class=value7>R</span><span class=value6>P</span><span class=value8>E</span><span class=value6>AQHR</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>E</span><span class=value9>KR</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------</span><span class=value6>CE</span><span class=valuegap>--</span><span class=value6>KP</span><span class=value9>R</span><span class=valuegap>-------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ER</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value5>P</span><span class=value6>Q</span><span class=valuegap>---</span><span class=value6>N</span><span class=valuegap>-</span><span class=value6>LCE</span><span class=valuegap>--</span><span class=value6>KPRR</span><span class=valuegap>------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ERRKHLFKQ</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ER</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ER</span><span class=value9>R</span><span class=value8>KHLFKQ</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value8>H</span><span class=value7>C</span><span class=valuegap>---</span><span class=value7>E</span><span class=valuegap>-</span><span class=value8>P</span><span class=value7>CS</span><span class=valuegap>--</span><span class=value7>EKRKH</span><span class=value8>LFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ER</span><span class=value9>R</span><span class=value8>KHLFKQ</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--</span><span class=value9>PP</span><span class=valuegap>------------</span><span class=value5>PG</span><span class=value1>W</span><span class=value9>F</span><span class=value5>YHHPN</span><span class=value4>SYH</span><span class=value5>T</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value6>T</span><span class=value7>V</span><span class=valuegap>---</span><span class=value7>N</span><span class=valuegap>-</span><span class=value7>LAQ</span><span class=valuegap>--</span><span class=value7>KGESTCSNK</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ER</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>EK</span><span class=value9>R</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------</span><span class=value7>H</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>E</span><span class=valuegap>-</span><span class=value8>P</span><span class=value9>CS</span><span class=valuegap>--</span><span class=value9>EKR</span><span class=value8>KHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------</span><span class=value6>P</span><span class=valuegap>-</span><span class=value6>P</span><span class=value7>CL</span><span class=valuegap>--</span><span class=value7>DRRKQ</span><span class=valuegap>---</span><span class=value7>P</span><span class=valuegap>-------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------------</span><span class=value6>A</span><span class=valuegap>-</span><span class=value6>PCP</span><span class=valuegap>--</span><span class=value6>DRKKR</span><span class=valuegap>---</span><span class=value7>V</span><span class=valuegap>-------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=valuegap>---</span><span class=value5>S</span><span class=valuegap>-</span><span class=value6>PRA</span><span class=valuegap>--</span><span class=value6>FSWPNS</span><span class=value4>R</span><span class=value6>EH</span><span class=value4>RAG</span><span class=value5>G</span><span class=valuegap>---</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=valuegap>---</span><span class=value5>S</span><span class=valuegap>-</span><span class=value6>PRA</span><span class=valuegap>--</span><span class=value6>FSWPNS</span><span class=value4>R</span><span class=value6>EH</span><span class=value4>R</span><span class=value3>A</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=valuegap>---</span><span class=value5>S</span><span class=valuegap>-</span><span class=value6>PRA</span><span class=valuegap>--</span><span class=value6>FSWPNS</span><span class=value4>R</span><span class=value6>EH</span><span class=value4>RAG</span><span class=value5>G</span><span class=valuegap>---</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=valuegap>---</span><span class=value5>S</span><span class=valuegap>-</span><span class=value6>PRA</span><span class=valuegap>--</span><span class=value6>FSWPNS</span><span class=value4>R</span><span class=value6>EH</span><span class=value4>R</span><span class=value3>GGP</span><span class=valuegap>---</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=valuegap>---</span><span class=value6>S</span><span class=valuegap>-</span><span class=value6>PRP</span><span class=valuegap>--</span><span class=value6>LSRAHS</span><span class=value5>H</span><span class=value6>E</span><span class=value7>H</span><span class=value4>RAG</span><span class=value5>G</span><span class=valuegap>---</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=value2>SP</span><span class=valuegap>-</span><span class=value2>R</span><span class=valuegap>-</span><span class=value2>PLS</span><span class=valuegap>--</span><span class=value2>QARSHEHR</span><span class=value3>AAPRL</span><span class=valuegap>---</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value4>EET</span><span class=value1>DVR</span><span class=valuegap>---------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value1>PS</span><span class=value9>G</span><span class=value1>FESA</span><span class=value0>D</span><span class=valuegap>-------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value7>F</span><span class=value6>FS</span><span class=value5>P</span><span class=value4>W</span><span class=value5>Q</span><span class=value6>PRE</span><span class=valuegap>--</span><span class=value6>VIRLFL</span><span class=value4>F</span><span class=valuegap>---------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>R</span><span class=valuegap>---</span><span class=value6>Q</span><span class=valuegap>-</span><span class=value6>VHS</span><span class=valuegap>--</span><span class=value6>IIRRSLPAT</span><span class=value3>LP</span><span class=valuegap>-</span><span class=value4>QCQA</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>R</span><span class=valuegap>---</span><span class=value6>H</span><span class=valuegap>-</span><span class=value6>PYS</span><span class=valuegap>--</span><span class=value6>IIRRSI</span><span class=value5>Q</span><span class=value6>IP</span><span class=value4>EE</span><span class=value3>D</span><span class=value4>RC</span><span class=value3>SH</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value3>PW</span><span class=value9>SLP</span><span class=value3>GPH</span><span class=value7>P</span><span class=value8>C</span><span class=valuegap>---</span><span class=value8>G</span><span class=valuegap>-</span><span class=value8>PCS</span><span class=valuegap>--</span><span class=value8>ERRKHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-------</span><span class=value2>N</span><span class=value5>P</span><span class=value7>C</span><span class=valuegap>---</span><span class=value7>G</span><span class=valuegap>-</span><span class=value7>P</span><span class=value8>CS</span><span class=valuegap>--</span><span class=value8>ERRKHLFVQ</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------</span><span class=value6>HHR</span><span class=valuegap>---</span><span class=value6>P</span><span class=valuegap>-------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>C</span><span class=valuegap>---</span><span class=value6>P</span><span class=valuegap>-</span><span class=value6>RCT</span><span class=valuegap>--</span><span class=value6>QH</span><span class=value7>H</span><span class=value6>QR</span><span class=valuegap>---</span><span class=value8>P</span><span class=valuegap>-------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------</span><span class=value1>RG</span><span class=valuegap>--</span><span class=value1>G</span><span class=valuegap>---------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>--------------</span><span class=value1>T</span><span class=value2>SLN</span><span class=valuegap>--</span><span class=value2>PDYREED</span><span class=value6>T</span><span class=value7>G</span><span class=value5>R</span><span class=value6>P</span><span class=value5>RE</span><span class=valuegap>---</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>V</span><span class=value2>A</span><span class=value6>A</span><span class=value4>AR</span><span class=valuegap>-</span><span class=value6>PVT</span><span class=valuegap>--</span><span class=value6>RSPGGS</span><span class=value5>Q</span><span class=value6>EQ</span><span class=value4>RAK</span><span class=value5>T</span><span class=valuegap>---</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value2>C</span><span class=valuegap>------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;</span><span class=value1>&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value2>&nbsp;</span><span class=value1>&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value1>&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value0>&nbsp;</span><span class=value9>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKNHI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNQIC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSYLGDT</span><span class=value8>DT</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKNHI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNQIC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSYLGDT</span><span class=value8>DT</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKSYI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNRIC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSHTGDT</span><span class=value8>DS</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKSYI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNHMC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSHLGDT</span><span class=value8>DS</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKNHI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNQIC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSYLGDT</span><span class=value8>DT</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PKNHI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>NNQIC</span><span class=value5>RCLAQH</span><span class=value6>D</span><span class=value9>FSFSSYLGDT</span><span class=value8>DT</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>FKKLC</span><span class=value5>HHGWI</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>DV</span><span class=value5>HKCECVENED</span><span class=valuegap>--------</span><span class=value8>HP</span><span class=valuegap>--</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value4>LKKLC</span><span class=value5>HNGWI</span><span class=valuegap>-</span><span class=value5>WDINKCECVQ</span><span class=value4>N</span><span class=value5>ED</span><span class=valuegap>--------</span><span class=value8>HP</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value3>IN</span><span class=value4>KLC</span><span class=value5>RSGWM</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>DS</span><span class=value5>DKCECVV</span><span class=value4>DK</span><span class=value5>D</span><span class=valuegap>--------</span><span class=value8>HP</span><span class=valuegap>--</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value4>INKLC</span><span class=value5>RSGWM</span><span class=valuegap>-</span><span class=value5>W</span><span class=value6>DS</span><span class=value5>DKCECVV</span><span class=value4>D</span><span class=value5>KD</span><span class=valuegap>--------</span><span class=value8>HP</span><span class=valuegap>--</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>PPLSTRA</span><span class=value4>SL</span><span class=value3>P</span><span class=value4>ARARAHRLLPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>QE</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------</span><span class=value4>RARAHRLLPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>QE</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value4>P</span><span class=value3>PLSTRA</span><span class=value4>SL</span><span class=value3>P</span><span class=value4>ARARAHRLLPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>QE</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value2>PLSTRAS</span><span class=valuegap>-</span><span class=value4>L</span><span class=value3>P</span><span class=value4>ARARAHRLLPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>QE</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>PRLTTVG</span><span class=value4>PL</span><span class=value3>Q</span><span class=value4>ARARAHRLMPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>HD</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value2>STVGP</span><span class=valuegap>---</span><span class=value5>L</span><span class=value4>P</span><span class=value5>ARA</span><span class=value4>RAHRLMPQ</span><span class=value5>K</span><span class=value4>R</span><span class=valuegap>--</span><span class=value7>KHRKFK</span><span class=value3>H</span><span class=value4>V</span><span class=value3>HD</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>ANKTC</span><span class=value5>PTNYM</span><span class=valuegap>-</span><span class=value5>WNNHICRCLAQED</span><span class=value9>FMFSSDAGD</span><span class=valuegap>-</span><span class=value6>DS</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>SKKLC</span><span class=value5>PIDML</span><span class=valuegap>-</span><span class=value5>WDSNKCKCVL</span><span class=value4>QEE</span><span class=valuegap>--------</span><span class=value7>NP</span><span class=valuegap>--</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value5>S</span><span class=value4>GKKRKR</span><span class=valuegap>----------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><span class=value3>PQ</span><span class=value2>TRVTI</span><span class=value6>R</span><span class=valuegap>-----</span><span class=value5>TVRVRRPPKG</span><span class=valuegap>--</span><span class=value4>K</span><span class=value5>H</span><span class=value4>RKFK</span><span class=value3>H</span><span class=value4>T</span><span class=value3>HD</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value3>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value2>&nbsp;</span><span class=value4>&nbsp;</span><span class=value5>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value4>&nbsp;</span><span class=value5>&nbsp;</span><span class=value4>&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value5>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value8>AEGY</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGPHK</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value8>AEGY</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGPHK</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>AEGF</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGAHK</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value8>TEGF</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGAHK</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value8>AEGY</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGPHK</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value8>AEGY</span><span class=value7>HDICGP</span><span class=value9>NKELDEETCQCVCKGGVRPSSCGPHK</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=value3>KKAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=value3>KKAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=value3>KKAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=value3>KKAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=value3>KKVL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value3>KQAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>TDGFHDICGP</span><span class=value9>NKELDEETCQCVCRAGLRPASCGPHK</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value3>KTAL</span><span class=valuegap>--------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;&nbsp;</span><span class=value4>&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCV</span><span class=value8>CK</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCV</span><span class=value8>CK</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCQCTCKNKLLPSSCGPNRDFDEERCHCV</span><span class=value8>CK</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCECTCKNKLLPSSCGPNKDFDEERCHCV</span><span class=value8>CK</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCV</span><span class=value8>CK</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRTSCQCTCKNKLLPSSCGPSKEFDEEKCQCV</span><span class=value8>CK</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value5>SRREGLPPLAEL</span><span class=value9>AMCAPNMDFDEENCECI</span><span class=value8>CK</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value5>SRREGLPPLAEL</span><span class=value9>AMCAPNMDFDEENCECV</span><span class=value8>CK</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value5>GRREGLPPLAEL</span><span class=value9>AMCAPNMNFDEENCECI</span><span class=value8>CK</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value5>GRREGLPPLAEL</span><span class=value9>AMCAPNMNFDEENCECI</span><span class=value8>CK</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>ELDRNSCQCVCKNKLFPSQCGANREFDENTCQCVCK</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----</span><span class=value8>L</span><span class=value5>AGTEDHSHLQEP</span><span class=value9>ALCGPHMMFDEDRCECV</span><span class=value8>CK</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value8>RTCPKHQPLNSAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>NKC</span><span class=value8>F</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value8>RTCPKHQPLNSAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>NKC</span><span class=value8>F</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>RTCPKHQPLNPAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>NKC</span><span class=value8>F</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value8>KTCPKHQPLNPAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>N</span><span class=value8>R</span><span class=value7>C</span><span class=value8>F</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value8>RTCPKHQPLNSAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>N</span><span class=value8>K</span><span class=value7>C</span><span class=value8>F</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value8>RTCPKHQPLNSAKC</span><span class=valuegap>---------</span><span class=value8>IC</span><span class=value7>ECVES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>N</span><span class=value8>K</span><span class=value7>C</span><span class=value8>F</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>WKCTGNLFQ</span><span class=value7>NKENC</span><span class=valuegap>---------</span><span class=value6>TC</span><span class=value7>YLCTENQ</span><span class=value6>DSCF</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value8>WKCTGNLFQNKENC</span><span class=valuegap>---------</span><span class=value8>SC</span><span class=value7>Y</span><span class=value6>LCTE</span><span class=value4>N</span><span class=value5>Q</span><span class=value8>DS</span><span class=value7>C</span><span class=value8>F</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value8>WKCTGNLFQNKENC</span><span class=valuegap>---------</span><span class=value8>SC</span><span class=value7>Y</span><span class=value6>LCTE</span><span class=value4>N</span><span class=value5>Q</span><span class=value8>DS</span><span class=value7>C</span><span class=value8>L</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value8>WKCTGNLFQNKENC</span><span class=valuegap>---------</span><span class=value8>SC</span><span class=value7>Y</span><span class=value6>LCTE</span><span class=value4>N</span><span class=value5>Q</span><span class=value7>D</span><span class=value8>S</span><span class=value7>C</span><span class=value8>L</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value4>DQ</span><span class=value5>E</span><span class=value4>E</span><span class=valuegap>------</span><span class=value9>AT</span><span class=valuegap>----------</span><span class=value4>NLPLS</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value4>DQ</span><span class=value5>E</span><span class=value4>E</span><span class=valuegap>------</span><span class=value9>AT</span><span class=valuegap>----------</span><span class=value4>NLPLS</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value4>DQ</span><span class=value5>E</span><span class=value4>E</span><span class=valuegap>------</span><span class=value9>AT</span><span class=valuegap>----------</span><span class=value4>NLPLS</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKYT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKYT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKYT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value7>DAQTC</span><span class=valuegap>---------</span><span class=value8>KCSCKFT</span><span class=valuegap>-</span><span class=value7>DA</span><span class=value6>R</span><span class=value7>C</span><span class=value5>K</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value3>V</span><span class=value4>H</span><span class=valuegap>-------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value6>I</span><span class=value7>P</span><span class=value6>RP</span><span class=value5>V</span><span class=valuegap>---------</span><span class=value6>NVPAN</span><span class=value7>SQ</span><span class=valuegap>-</span><span class=value6>THV</span><span class=value5>AS</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>D</span><span class=value8>A</span><span class=value7>RCK</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>DARCK</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DAQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKFT</span><span class=valuegap>-</span><span class=value7>D</span><span class=value8>A</span><span class=value7>RCK</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value7>DP</span><span class=value8>E</span><span class=value7>T</span><span class=valuegap>---------</span><span class=value1>C</span><span class=value7>KCH</span><span class=value8>CRRR</span><span class=valuegap>-</span><span class=value6>P</span><span class=value7>QRC</span><span class=value6>R</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value7>NP</span><span class=value8>V</span><span class=value7>T</span><span class=valuegap>---------</span><span class=value2>C</span><span class=value7>ECV</span><span class=value9>CKR</span><span class=value8>D</span><span class=valuegap>-</span><span class=value6>S</span><span class=value7>VRC</span><span class=value6>Q</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>RTC</span><span class=value8>PRNQPLNPGKC</span><span class=valuegap>---------</span><span class=value8>AC</span><span class=value7>ECTES</span><span class=value5>P</span><span class=valuegap>-</span><span class=value7>QKC</span><span class=value8>L</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>TPCPKDLIQHPKNC</span><span class=valuegap>---------</span><span class=value8>SC</span><span class=value7>F</span><span class=value6>ECKE</span><span class=value4>S</span><span class=value5>L</span><span class=value7>ETC</span><span class=value8>C</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKNT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value8>DPQT</span><span class=value7>C</span><span class=valuegap>---------</span><span class=value8>KCS</span><span class=value9>CKNT</span><span class=valuegap>-</span><span class=value7>DSRCK</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value4>QP</span><span class=value5>RS</span><span class=value2>V</span><span class=value3>PGWDS</span><span class=value9>AP</span><span class=value3>G</span><span class=value2>A</span><span class=value5>PSP</span><span class=value6>ADIT</span><span class=value4>H</span><span class=value5>PTPA</span><span class=value4>P</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>---------</span><span class=value7>DP</span><span class=value8>R</span><span class=value7>T</span><span class=valuegap>---------</span><span class=value1>C</span><span class=value7>RCR</span><span class=value8>CRRR</span><span class=value9>S</span><span class=value6>F</span><span class=value7>LRC</span><span class=value6>Q</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------------------------------</span><span class=value2>LC</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value8>&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><span class=value0>&nbsp;</span><span class=value1>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>&nbsp;</span><span class=value6>&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>NHQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>NHQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>HQQTC</span><span class=value8>SC</span><span class=value9>Y</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>HQQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>NHQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>R</span><span class=value8>F</span><span class=value7>NHQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RP</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>QKHKTFNAETC</span><span class=value9>SC</span><span class=value8>E</span><span class=value9>DK</span><span class=value5>C</span><span class=value6>PFQARTCPTA</span><span class=value5>RPAC</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=value8>QKHK</span><span class=value7>T</span><span class=value8>FNAE</span><span class=value7>TC</span><span class=value8>SC</span><span class=value7>E</span><span class=value8>DK</span><span class=value6>CPFQARTCPTARP</span><span class=value7>A</span><span class=value5>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=value8>Q</span><span class=value7>KH</span><span class=value8>K</span><span class=value7>E</span><span class=value8>FNAE</span><span class=value7>TC</span><span class=value8>SC</span><span class=value7>E</span><span class=value8>DK</span><span class=value6>CPFQARMCPTARP</span><span class=value7>A</span><span class=value5>C</span><span class=valuegap>-----</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=value8>QKHK</span><span class=value7>E</span><span class=value8>FNAE</span><span class=value7>TC</span><span class=value8>SC</span><span class=value7>E</span><span class=value8>DK</span><span class=value6>CPFQARMCPTARP</span><span class=value7>A</span><span class=value5>C</span><span class=valuegap>-----</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=value3>ERDV</span><span class=value4>P</span><span class=value6>L</span><span class=valuegap>------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=value3>ERDV</span><span class=value4>P</span><span class=value6>L</span><span class=valuegap>------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=value3>ERDV</span><span class=value4>P</span><span class=value6>L</span><span class=valuegap>------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value9>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=value7>SRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=value7>SRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=value7>SRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=value7>SRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>LRQL</span><span class=value7>E</span><span class=value5>LNERTC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value6>K</span><span class=value7>PR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=value7>SRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value9>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=value4>RGSL</span><span class=value5>S</span><span class=valuegap>-------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>SKQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>TC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>LRQL</span><span class=value8>E</span><span class=value7>LNER</span><span class=value6>T</span><span class=value7>CRCE</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>NLGL</span><span class=value8>E</span><span class=value6>LNEHSC</span><span class=value7>RCE</span><span class=valuegap>-------------</span><span class=value7>KLR</span><span class=value2>R</span><span class=valuegap>-----</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>KGN</span><span class=valuegap>-------------</span><span class=value5>AEG</span><span class=valuegap>------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>KGN</span><span class=valuegap>-------------</span><span class=value5>AEG</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=value6>QRGW</span><span class=value8>E</span><span class=value6>LNKATC</span><span class=value7>RCV</span><span class=valuegap>-------------</span><span class=value7>KH</span><span class=value6>R</span><span class=value2>R</span><span class=value1>R</span><span class=valuegap>----</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>KGN</span><span class=valuegap>-------------</span><span class=value5>TEG</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KEI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KEI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KEI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KEI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KDI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>SSR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>SSR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>SSR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>SSR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KDI</span><span class=valuegap>-------------</span><span class=value7>FPA</span><span class=valuegap>------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>SAR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>L</span><span class=value7>KG</span><span class=value8>K</span><span class=value7>K</span><span class=value8>F</span><span class=value7>HHQTC</span><span class=value8>SCY</span><span class=valuegap>-------------</span><span class=value7>RR</span><span class=value8>P</span><span class=value6>C</span><span class=valuegap>-----</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value8>Q</span><span class=value7>KH</span><span class=value8>K</span><span class=value7>L</span><span class=value8>FH</span><span class=value7>P</span><span class=value8>D</span><span class=value7>TC</span><span class=value8>SC</span><span class=value7>E</span><span class=value8>DR</span><span class=value7>CPFHTRPCASGKTA</span><span class=value5>C</span><span class=valuegap>-----</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>ARQL</span><span class=value8>E</span><span class=value7>L</span><span class=value6>NE</span><span class=value7>R</span><span class=value6>TC</span><span class=value7>RCD</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>ARQL</span><span class=value8>E</span><span class=value7>L</span><span class=value6>NE</span><span class=value7>R</span><span class=value6>TC</span><span class=value7>RCD</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KCD</span><span class=valuegap>-------------</span><span class=value7>KPR</span><span class=valuegap>------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value4>GPSA</span><span class=value5>H</span><span class=value4>AAPSTTS</span><span class=value5>A</span><span class=value4>L</span><span class=valuegap>-------------</span><span class=value3>TPG</span><span class=value1>PAAAAA</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>GRGL</span><span class=value7>E</span><span class=value6>LNPDTC</span><span class=value7>RCR</span><span class=valuegap>-------------</span><span class=value7>KL</span><span class=valuegap>-------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>GDAVPR</span><span class=value2>R</span><span class=valuegap>-----------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value4>V</span><span class=value5>R</span><span class=value8>AK</span><span class=valuegap>-------------</span><span class=value8>F</span><span class=value5>P</span><span class=valuegap>-------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value8>KR</span><span class=value7>L</span><span class=valuegap>-------------</span><span class=value7>KP</span><span class=value5>T</span><span class=valuegap>------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>KET</span><span class=valuegap>-------------</span><span class=value7>LGA</span><span class=valuegap>------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value4>RGS</span><span class=valuegap>-------------</span><span class=value5>TGG</span><span class=valuegap>------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value5>SSR</span><span class=valuegap>-------------</span><span class=value5>PPR</span><span class=valuegap>------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value7>&nbsp;</span><span class=value6>&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;</span><span class=value8>&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRTKRCEPGFYYSEEVCRCVP</span><span class=value7>TYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRTKRCEPGFYYSEEVCRCVP</span><span class=value7>TYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRMKRCESGFYYSEEVCRCVP</span><span class=value7>IYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRTKRCESGFYYSEEVCRCVP</span><span class=value7>IYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRTKRCEPGFYYSEEVCRCVP</span><span class=value7>TYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value8>TVRTKRCEPGFYYSEEVCRCVP</span><span class=value7>TYWKRP</span><span class=value9>LI</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value5>SRHCRCL</span><span class=value9>RGGRSPHG</span><span class=value5>SQSREN</span><span class=value6>P</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value6>SRHCRCL</span><span class=value9>RGGRGHHG</span><span class=value6>SQSKEN</span><span class=value7>P</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value6>SRHCCRP</span><span class=value9>RGGRSPHG</span><span class=value6>SQSRET</span><span class=value7>P</span><span class=valuegap>--------</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value6>SRHCCRP</span><span class=value9>RGGRSPHG</span><span class=value6>SQSRET</span><span class=value7>P</span><span class=valuegap>--------</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value1>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value7>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value5>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----</span><span class=value2>S</span><span class=value6>GRKA</span><span class=value5>KA</span><span class=valuegap>------------------------</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value6>TNRQKAC</span><span class=value7>EPGFSYSE</span><span class=value6>EVCRCVPSYWKRP</span><span class=value9>QM</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------</span><span class=value7>AKHCRFP</span><span class=value9>KEKRAAQG</span><span class=value7>PHSRKN</span><span class=value6>P</span><span class=valuegap>--------</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-----------</span><span class=value6>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value1>DAAASSVAKGG</span><span class=value4>A</span><span class=valuegap>------------------------</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value1>R</span><span class=value5>R</span><span class=valuegap>------------------------</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>----------</span><span class=value1>F</span><span class=value3>V</span><span class=valuegap>------------------------</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>------------------------------------</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value0>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value6>&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value5>&nbsp;</span><span class=value6>&nbsp;</span><span class=value7>&nbsp;</span><span class=value8>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value7>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;&nbsp;</span><br><br><br><span class=valuedefault>XP_028599647.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>XP_028599646.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>GCF50945.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>XP_015283812.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>XP_028599648.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>XP_028599649.1&nbsp;&nbsp;&nbsp;</span><span class=value9>N</span><br><span class=valuedefault>GCF40285.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015273092.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028583464.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028583463.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADJ67670.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADJ67669.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028597747.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028597745.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028597744.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028575451.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03699.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03700.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03693.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015284721.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03701.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015276119.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015284720.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028580181.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028580180.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF49643.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015284719.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32848.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028580177.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03692.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028580176.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015284783.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32850.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03694.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32847.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028580178.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03695.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF54425.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03698.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03697.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>ADB03696.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32851.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32852.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>AJR32849.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015272314.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF46055.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF40753.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>NP_001310426.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028566508.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028599432.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028601741.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028601742.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028601735.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028601740.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015274676.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285151.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF41376.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285149.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015285150.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF46860.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>GCF52154.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028582005.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_015272715.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>XP_028561875.1&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>S</span><br><span class=valuedefault>VEGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-A206&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-A165&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-A121&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-B186&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-B167&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>PlGF-1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>VEGF-F&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>PDGF-A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>PDGF-B&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>PDGF-C&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><span class=valuedefault>PDGF-D&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=valuegap>-</span><br><br><span class=valuedefault>cons&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class=value9>&nbsp;</span><br><br><br><br><br><br></body></html>

</pre>
</body>
</html
>